// angular-feather/esm2015/index.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
export { IconActivity } from './feather/activity';
export { IconAirplay } from './feather/airplay';
export { IconAlertCircle } from './feather/alert-circle';
export { IconAlertOctagon } from './feather/alert-octagon';
export { IconAlertTriangle } from './feather/alert-triangle';
export { IconAlignCenter } from './feather/align-center';
export { IconAlignJustify } from './feather/align-justify';
export { IconAlignLeft } from './feather/align-left';
export { IconAlignRight } from './feather/align-right';
export { IconAnchor } from './feather/anchor';
export { IconAperture } from './feather/aperture';
export { IconArchive } from './feather/archive';
export { IconArrowDownCircle } from './feather/arrow-down-circle';
export { IconArrowDownLeft } from './feather/arrow-down-left';
export { IconArrowDownRight } from './feather/arrow-down-right';
export { IconArrowDown } from './feather/arrow-down';
export { IconArrowLeftCircle } from './feather/arrow-left-circle';
export { IconArrowLeft } from './feather/arrow-left';
export { IconArrowRightCircle } from './feather/arrow-right-circle';
export { IconArrowRight } from './feather/arrow-right';
export { IconArrowUpCircle } from './feather/arrow-up-circle';
export { IconArrowUpLeft } from './feather/arrow-up-left';
export { IconArrowUpRight } from './feather/arrow-up-right';
export { IconArrowUp } from './feather/arrow-up';
export { IconAtSign } from './feather/at-sign';
export { IconAward } from './feather/award';
export { IconBarChart2 } from './feather/bar-chart-2';
export { IconBarChart } from './feather/bar-chart';
export { IconBatteryCharging } from './feather/battery-charging';
export { IconBattery } from './feather/battery';
export { IconBellOff } from './feather/bell-off';
export { IconBell } from './feather/bell';
export { IconBluetooth } from './feather/bluetooth';
export { IconBold } from './feather/bold';
export { IconBookOpen } from './feather/book-open';
export { IconBook } from './feather/book';
export { IconBookmark } from './feather/bookmark';
export { IconBox } from './feather/box';
export { IconBriefcase } from './feather/briefcase';
export { IconCalendar } from './feather/calendar';
export { IconCameraOff } from './feather/camera-off';
export { IconCamera } from './feather/camera';
export { IconCast } from './feather/cast';
export { IconCheckCircle } from './feather/check-circle';
export { IconCheckSquare } from './feather/check-square';
export { IconCheck } from './feather/check';
export { IconChevronDown } from './feather/chevron-down';
export { IconChevronLeft } from './feather/chevron-left';
export { IconChevronRight } from './feather/chevron-right';
export { IconChevronUp } from './feather/chevron-up';
export { IconChevronsDown } from './feather/chevrons-down';
export { IconChevronsLeft } from './feather/chevrons-left';
export { IconChevronsRight } from './feather/chevrons-right';
export { IconChevronsUp } from './feather/chevrons-up';
export { IconChrome } from './feather/chrome';
export { IconCircle } from './feather/circle';
export { IconClipboard } from './feather/clipboard';
export { IconClock } from './feather/clock';
export { IconCloudDrizzle } from './feather/cloud-drizzle';
export { IconCloudLightning } from './feather/cloud-lightning';
export { IconCloudOff } from './feather/cloud-off';
export { IconCloudRain } from './feather/cloud-rain';
export { IconCloudSnow } from './feather/cloud-snow';
export { IconCloud } from './feather/cloud';
export { IconCode } from './feather/code';
export { IconCodepen } from './feather/codepen';
export { IconCoffee } from './feather/coffee';
export { IconCommand } from './feather/command';
export { IconCompass } from './feather/compass';
export { IconCopy } from './feather/copy';
export { IconCornerDownLeft } from './feather/corner-down-left';
export { IconCornerDownRight } from './feather/corner-down-right';
export { IconCornerLeftDown } from './feather/corner-left-down';
export { IconCornerLeftUp } from './feather/corner-left-up';
export { IconCornerRightDown } from './feather/corner-right-down';
export { IconCornerRightUp } from './feather/corner-right-up';
export { IconCornerUpLeft } from './feather/corner-up-left';
export { IconCornerUpRight } from './feather/corner-up-right';
export { IconCpu } from './feather/cpu';
export { IconCreditCard } from './feather/credit-card';
export { IconCrop } from './feather/crop';
export { IconCrosshair } from './feather/crosshair';
export { IconDatabase } from './feather/database';
export { IconDelete } from './feather/delete';
export { IconDisc } from './feather/disc';
export { IconDollarSign } from './feather/dollar-sign';
export { IconDownloadCloud } from './feather/download-cloud';
export { IconDownload } from './feather/download';
export { IconDroplet } from './feather/droplet';
export { IconEdit2 } from './feather/edit-2';
export { IconEdit3 } from './feather/edit-3';
export { IconEdit } from './feather/edit';
export { IconExternalLink } from './feather/external-link';
export { IconEyeOff } from './feather/eye-off';
export { IconEye } from './feather/eye';
export { IconFacebook } from './feather/facebook';
export { IconFastForward } from './feather/fast-forward';
export { IconFeather } from './feather/feather';
export { IconFigma } from './feather/figma';
export { IconFileMinus } from './feather/file-minus';
export { IconFilePlus } from './feather/file-plus';
export { IconFileText } from './feather/file-text';
export { IconFile } from './feather/file';
export { IconFilm } from './feather/film';
export { IconFilter } from './feather/filter';
export { IconFlag } from './feather/flag';
export { IconFolderMinus } from './feather/folder-minus';
export { IconFolderPlus } from './feather/folder-plus';
export { IconFolder } from './feather/folder';
export { IconFrown } from './feather/frown';
export { IconGift } from './feather/gift';
export { IconGitBranch } from './feather/git-branch';
export { IconGitCommit } from './feather/git-commit';
export { IconGitMerge } from './feather/git-merge';
export { IconGitPullRequest } from './feather/git-pull-request';
export { IconGithub } from './feather/github';
export { IconGitlab } from './feather/gitlab';
export { IconGlobe } from './feather/globe';
export { IconGrid } from './feather/grid';
export { IconHardDrive } from './feather/hard-drive';
export { IconHash } from './feather/hash';
export { IconHeadphones } from './feather/headphones';
export { IconHeart } from './feather/heart';
export { IconHelpCircle } from './feather/help-circle';
export { IconHome } from './feather/home';
export { IconImage } from './feather/image';
export { IconInbox } from './feather/inbox';
export { IconInfo } from './feather/info';
export { IconInstagram } from './feather/instagram';
export { IconItalic } from './feather/italic';
export { IconKey } from './feather/key';
export { IconLayers } from './feather/layers';
export { IconLayout } from './feather/layout';
export { IconLifeBuoy } from './feather/life-buoy';
export { IconLink2 } from './feather/link-2';
export { IconLink } from './feather/link';
export { IconLinkedin } from './feather/linkedin';
export { IconList } from './feather/list';
export { IconLoader } from './feather/loader';
export { IconLock } from './feather/lock';
export { IconLogIn } from './feather/log-in';
export { IconLogOut } from './feather/log-out';
export { IconMail } from './feather/mail';
export { IconMapPin } from './feather/map-pin';
export { IconMap } from './feather/map';
export { IconMaximize2 } from './feather/maximize-2';
export { IconMaximize } from './feather/maximize';
export { IconMeh } from './feather/meh';
export { IconMenu } from './feather/menu';
export { IconMessageCircle } from './feather/message-circle';
export { IconMessageSquare } from './feather/message-square';
export { IconMicOff } from './feather/mic-off';
export { IconMic } from './feather/mic';
export { IconMinimize2 } from './feather/minimize-2';
export { IconMinimize } from './feather/minimize';
export { IconMinusCircle } from './feather/minus-circle';
export { IconMinusSquare } from './feather/minus-square';
export { IconMinus } from './feather/minus';
export { IconMonitor } from './feather/monitor';
export { IconMoon } from './feather/moon';
export { IconMoreHorizontal } from './feather/more-horizontal';
export { IconMoreVertical } from './feather/more-vertical';
export { IconMousePointer } from './feather/mouse-pointer';
export { IconMove } from './feather/move';
export { IconMusic } from './feather/music';
export { IconNavigation2 } from './feather/navigation-2';
export { IconNavigation } from './feather/navigation';
export { IconOctagon } from './feather/octagon';
export { IconPackage } from './feather/package';
export { IconPaperclip } from './feather/paperclip';
export { IconPauseCircle } from './feather/pause-circle';
export { IconPause } from './feather/pause';
export { IconPenTool } from './feather/pen-tool';
export { IconPercent } from './feather/percent';
export { IconPhoneCall } from './feather/phone-call';
export { IconPhoneForwarded } from './feather/phone-forwarded';
export { IconPhoneIncoming } from './feather/phone-incoming';
export { IconPhoneMissed } from './feather/phone-missed';
export { IconPhoneOff } from './feather/phone-off';
export { IconPhoneOutgoing } from './feather/phone-outgoing';
export { IconPhone } from './feather/phone';
export { IconPieChart } from './feather/pie-chart';
export { IconPlayCircle } from './feather/play-circle';
export { IconPlay } from './feather/play';
export { IconPlusCircle } from './feather/plus-circle';
export { IconPlusSquare } from './feather/plus-square';
export { IconPlus } from './feather/plus';
export { IconPocket } from './feather/pocket';
export { IconPower } from './feather/power';
export { IconPrinter } from './feather/printer';
export { IconRadio } from './feather/radio';
export { IconRefreshCcw } from './feather/refresh-ccw';
export { IconRefreshCw } from './feather/refresh-cw';
export { IconRepeat } from './feather/repeat';
export { IconRewind } from './feather/rewind';
export { IconRotateCcw } from './feather/rotate-ccw';
export { IconRotateCw } from './feather/rotate-cw';
export { IconRss } from './feather/rss';
export { IconSave } from './feather/save';
export { IconScissors } from './feather/scissors';
export { IconSearch } from './feather/search';
export { IconSend } from './feather/send';
export { IconServer } from './feather/server';
export { IconSettings } from './feather/settings';
export { IconShare2 } from './feather/share-2';
export { IconShare } from './feather/share';
export { IconShieldOff } from './feather/shield-off';
export { IconShield } from './feather/shield';
export { IconShoppingBag } from './feather/shopping-bag';
export { IconShoppingCart } from './feather/shopping-cart';
export { IconShuffle } from './feather/shuffle';
export { IconSidebar } from './feather/sidebar';
export { IconSkipBack } from './feather/skip-back';
export { IconSkipForward } from './feather/skip-forward';
export { IconSlack } from './feather/slack';
export { IconSlash } from './feather/slash';
export { IconSliders } from './feather/sliders';
export { IconSmartphone } from './feather/smartphone';
export { IconSmile } from './feather/smile';
export { IconSpeaker } from './feather/speaker';
export { IconSquare } from './feather/square';
export { IconStar } from './feather/star';
export { IconStopCircle } from './feather/stop-circle';
export { IconSun } from './feather/sun';
export { IconSunrise } from './feather/sunrise';
export { IconSunset } from './feather/sunset';
export { IconTablet } from './feather/tablet';
export { IconTag } from './feather/tag';
export { IconTarget } from './feather/target';
export { IconTerminal } from './feather/terminal';
export { IconThermometer } from './feather/thermometer';
export { IconThumbsDown } from './feather/thumbs-down';
export { IconThumbsUp } from './feather/thumbs-up';
export { IconToggleLeft } from './feather/toggle-left';
export { IconToggleRight } from './feather/toggle-right';
export { IconTrash2 } from './feather/trash-2';
export { IconTrash } from './feather/trash';
export { IconTrello } from './feather/trello';
export { IconTrendingDown } from './feather/trending-down';
export { IconTrendingUp } from './feather/trending-up';
export { IconTriangle } from './feather/triangle';
export { IconTruck } from './feather/truck';
export { IconTv } from './feather/tv';
export { IconTwitter } from './feather/twitter';
export { IconType } from './feather/type';
export { IconUmbrella } from './feather/umbrella';
export { IconUnderline } from './feather/underline';
export { IconUnlock } from './feather/unlock';
export { IconUploadCloud } from './feather/upload-cloud';
export { IconUpload } from './feather/upload';
export { IconUserCheck } from './feather/user-check';
export { IconUserMinus } from './feather/user-minus';
export { IconUserPlus } from './feather/user-plus';
export { IconUserX } from './feather/user-x';
export { IconUser } from './feather/user';
export { IconUsers } from './feather/users';
export { IconVideoOff } from './feather/video-off';
export { IconVideo } from './feather/video';
export { IconVoicemail } from './feather/voicemail';
export { IconVolume1 } from './feather/volume-1';
export { IconVolume2 } from './feather/volume-2';
export { IconVolumeX } from './feather/volume-x';
export { IconVolume } from './feather/volume';
export { IconWatch } from './feather/watch';
export { IconWifiOff } from './feather/wifi-off';
export { IconWifi } from './feather/wifi';
export { IconWind } from './feather/wind';
export { IconXCircle } from './feather/x-circle';
export { IconXOctagon } from './feather/x-octagon';
export { IconXSquare } from './feather/x-square';
export { IconX } from './feather/x';
export { IconYoutube } from './feather/youtube';
export { IconZapOff } from './feather/zap-off';
export { IconZap } from './feather/zap';
export { IconZoomIn } from './feather/zoom-in';
export { IconZoomOut } from './feather/zoom-out';
