// angular-feather/esm2015/feather/alert-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconAlertCircleComponent {
}
IconAlertCircleComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-alert-circle',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-alert-circle">
    <circle cx="12" cy="12" r="10"></circle><line x1="12" y1="8" x2="12" y2="12"></line><line x1="12" y1="16" x2="12" y2="16"></line>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconAlertCircle {
}
IconAlertCircle.decorators = [
    { type: NgModule, args: [{
                declarations: [IconAlertCircleComponent],
                exports: [IconAlertCircleComponent]
            },] }
];
