// angular-feather/esm2015/feather/arrow-up.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconArrowUpComponent {
}
IconArrowUpComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-arrow-up',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-arrow-up">
    <line x1="12" y1="19" x2="12" y2="5"></line><polyline points="5 12 12 5 19 12"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconArrowUp {
}
IconArrowUp.decorators = [
    { type: NgModule, args: [{
                declarations: [IconArrowUpComponent],
                exports: [IconArrowUpComponent]
            },] }
];
