// angular-feather/esm2015/feather/arrow-down.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconArrowDownComponent {
}
IconArrowDownComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-arrow-down',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-arrow-down">
    <line x1="12" y1="5" x2="12" y2="19"></line><polyline points="19 12 12 19 5 12"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconArrowDown {
}
IconArrowDown.decorators = [
    { type: NgModule, args: [{
                declarations: [IconArrowDownComponent],
                exports: [IconArrowDownComponent]
            },] }
];
