// angular-feather/esm2015/feather/arrow-up-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconArrowUpCircleComponent {
}
IconArrowUpCircleComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-arrow-up-circle',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-arrow-up-circle">
    <circle cx="12" cy="12" r="10"></circle><polyline points="16 12 12 8 8 12"></polyline><line x1="12" y1="16" x2="12" y2="8"></line>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconArrowUpCircle {
}
IconArrowUpCircle.decorators = [
    { type: NgModule, args: [{
                declarations: [IconArrowUpCircleComponent],
                exports: [IconArrowUpCircleComponent]
            },] }
];
