// angular-feather/esm2015/feather/arrow-down-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconArrowDownCircleComponent {
}
IconArrowDownCircleComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-arrow-down-circle',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-arrow-down-circle">
    <circle cx="12" cy="12" r="10"></circle><polyline points="8 12 12 16 16 12"></polyline><line x1="12" y1="8" x2="12" y2="16"></line>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconArrowDownCircle {
}
IconArrowDownCircle.decorators = [
    { type: NgModule, args: [{
                declarations: [IconArrowDownCircleComponent],
                exports: [IconArrowDownCircleComponent]
            },] }
];
