// angular-feather/esm2015/feather/arrow-left-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconArrowLeftCircleComponent {
}
IconArrowLeftCircleComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-arrow-left-circle',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-arrow-left-circle">
    <circle cx="12" cy="12" r="10"></circle><polyline points="12 8 8 12 12 16"></polyline><line x1="16" y1="12" x2="8" y2="12"></line>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconArrowLeftCircle {
}
IconArrowLeftCircle.decorators = [
    { type: NgModule, args: [{
                declarations: [IconArrowLeftCircleComponent],
                exports: [IconArrowLeftCircleComponent]
            },] }
];
