// angular-feather/esm2015/feather/arrow-down-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconArrowDownLeftComponent {
}
IconArrowDownLeftComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-arrow-down-left',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-arrow-down-left">
    <line x1="17" y1="7" x2="7" y2="17"></line><polyline points="17 17 7 17 7 7"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconArrowDownLeft {
}
IconArrowDownLeft.decorators = [
    { type: NgModule, args: [{
                declarations: [IconArrowDownLeftComponent],
                exports: [IconArrowDownLeftComponent]
            },] }
];
