// angular-feather/esm2015/feather/arrow-up-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconArrowUpRightComponent {
}
IconArrowUpRightComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-arrow-up-right',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-arrow-up-right">
    <line x1="7" y1="17" x2="17" y2="7"></line><polyline points="7 7 17 7 17 17"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconArrowUpRight {
}
IconArrowUpRight.decorators = [
    { type: NgModule, args: [{
                declarations: [IconArrowUpRightComponent],
                exports: [IconArrowUpRightComponent]
            },] }
];
