// angular-feather/esm2015/feather/bluetooth.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconBluetoothComponent {
}
IconBluetoothComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-bluetooth',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-bluetooth">
    <polyline points="6.5 6.5 17.5 17.5 12 23 12 1 17.5 6.5 6.5 17.5"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconBluetooth {
}
IconBluetooth.decorators = [
    { type: NgModule, args: [{
                declarations: [IconBluetoothComponent],
                exports: [IconBluetoothComponent]
            },] }
];
