// angular-feather/esm2015/feather/check.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconCheckComponent {
}
IconCheckComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-check',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-check">
    <polyline points="20 6 9 17 4 12"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconCheck {
}
IconCheck.decorators = [
    { type: NgModule, args: [{
                declarations: [IconCheckComponent],
                exports: [IconCheckComponent]
            },] }
];
