// angular-feather/esm2015/feather/chevron-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconChevronRightComponent {
}
IconChevronRightComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-chevron-right',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-chevron-right">
    <polyline points="9 18 15 12 9 6"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconChevronRight {
}
IconChevronRight.decorators = [
    { type: NgModule, args: [{
                declarations: [IconChevronRightComponent],
                exports: [IconChevronRightComponent]
            },] }
];
