// angular-feather/esm2015/feather/chevron-up.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconChevronUpComponent {
}
IconChevronUpComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-chevron-up',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-chevron-up">
    <polyline points="18 15 12 9 6 15"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconChevronUp {
}
IconChevronUp.decorators = [
    { type: NgModule, args: [{
                declarations: [IconChevronUpComponent],
                exports: [IconChevronUpComponent]
            },] }
];
