// angular-feather/esm2015/feather/chevron-down.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconChevronDownComponent {
}
IconChevronDownComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-chevron-down',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-chevron-down">
    <polyline points="6 9 12 15 18 9"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconChevronDown {
}
IconChevronDown.decorators = [
    { type: NgModule, args: [{
                declarations: [IconChevronDownComponent],
                exports: [IconChevronDownComponent]
            },] }
];
