// angular-feather/esm2015/feather/chevrons-down.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconChevronsDownComponent {
}
IconChevronsDownComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-chevrons-down',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-chevrons-down">
    <polyline points="7 13 12 18 17 13"></polyline><polyline points="7 6 12 11 17 6"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconChevronsDown {
}
IconChevronsDown.decorators = [
    { type: NgModule, args: [{
                declarations: [IconChevronsDownComponent],
                exports: [IconChevronsDownComponent]
            },] }
];
