// angular-feather/esm2015/feather/chevrons-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconChevronsLeftComponent {
}
IconChevronsLeftComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-chevrons-left',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-chevrons-left">
    <polyline points="11 17 6 12 11 7"></polyline><polyline points="18 17 13 12 18 7"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconChevronsLeft {
}
IconChevronsLeft.decorators = [
    { type: NgModule, args: [{
                declarations: [IconChevronsLeftComponent],
                exports: [IconChevronsLeftComponent]
            },] }
];
