// angular-feather/esm2015/feather/chevrons-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconChevronsRightComponent {
}
IconChevronsRightComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-chevrons-right',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-chevrons-right">
    <polyline points="13 17 18 12 13 7"></polyline><polyline points="6 17 11 12 6 7"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconChevronsRight {
}
IconChevronsRight.decorators = [
    { type: NgModule, args: [{
                declarations: [IconChevronsRightComponent],
                exports: [IconChevronsRightComponent]
            },] }
];
