// angular-feather/esm2015/feather/cloud-lightning.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconCloudLightningComponent {
}
IconCloudLightningComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-cloud-lightning',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-cloud-lightning">
    <path d="M19 16.9A5 5 0 0 0 18 7h-1.26a8 8 0 1 0-11.62 9"></path><polyline points="13 11 9 17 15 17 11 23"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconCloudLightning {
}
IconCloudLightning.decorators = [
    { type: NgModule, args: [{
                declarations: [IconCloudLightningComponent],
                exports: [IconCloudLightningComponent]
            },] }
];
