// angular-feather/esm2015/feather/corner-right-up.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconCornerRightUpComponent {
}
IconCornerRightUpComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-corner-right-up',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-corner-right-up">
    <polyline points="10 9 15 4 20 9"></polyline><path d="M4 20h7a4 4 0 0 0 4-4V4"></path>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconCornerRightUp {
}
IconCornerRightUp.decorators = [
    { type: NgModule, args: [{
                declarations: [IconCornerRightUpComponent],
                exports: [IconCornerRightUpComponent]
            },] }
];
