// angular-feather/esm2015/feather/corner-down-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconCornerDownLeftComponent {
}
IconCornerDownLeftComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-corner-down-left',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-corner-down-left">
    <polyline points="9 10 4 15 9 20"></polyline><path d="M20 4v7a4 4 0 0 1-4 4H4"></path>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconCornerDownLeft {
}
IconCornerDownLeft.decorators = [
    { type: NgModule, args: [{
                declarations: [IconCornerDownLeftComponent],
                exports: [IconCornerDownLeftComponent]
            },] }
];
