// angular-feather/esm2015/feather/corner-up-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconCornerUpRightComponent {
}
IconCornerUpRightComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-corner-up-right',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-corner-up-right">
    <polyline points="15 14 20 9 15 4"></polyline><path d="M4 20v-7a4 4 0 0 1 4-4h12"></path>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconCornerUpRight {
}
IconCornerUpRight.decorators = [
    { type: NgModule, args: [{
                declarations: [IconCornerUpRightComponent],
                exports: [IconCornerUpRightComponent]
            },] }
];
