// angular-feather/esm2015/feather/corner-up-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconCornerUpLeftComponent {
}
IconCornerUpLeftComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-corner-up-left',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-corner-up-left">
    <polyline points="9 14 4 9 9 4"></polyline><path d="M20 20v-7a4 4 0 0 0-4-4H4"></path>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconCornerUpLeft {
}
IconCornerUpLeft.decorators = [
    { type: NgModule, args: [{
                declarations: [IconCornerUpLeftComponent],
                exports: [IconCornerUpLeftComponent]
            },] }
];
