// angular-feather/esm2015/feather/corner-left-up.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconCornerLeftUpComponent {
}
IconCornerLeftUpComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-corner-left-up',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-corner-left-up">
    <polyline points="14 9 9 4 4 9"></polyline><path d="M20 20h-7a4 4 0 0 1-4-4V4"></path>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconCornerLeftUp {
}
IconCornerLeftUp.decorators = [
    { type: NgModule, args: [{
                declarations: [IconCornerLeftUpComponent],
                exports: [IconCornerLeftUpComponent]
            },] }
];
