// angular-feather/esm2015/feather/corner-left-down.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconCornerLeftDownComponent {
}
IconCornerLeftDownComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-corner-left-down',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-corner-left-down">
    <polyline points="14 15 9 20 4 15"></polyline><path d="M20 4h-7a4 4 0 0 0-4 4v12"></path>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconCornerLeftDown {
}
IconCornerLeftDown.decorators = [
    { type: NgModule, args: [{
                declarations: [IconCornerLeftDownComponent],
                exports: [IconCornerLeftDownComponent]
            },] }
];
