// angular-feather/esm2015/feather/credit-card.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconCreditCardComponent {
}
IconCreditCardComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-credit-card',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-credit-card">
    <rect x="1" y="4" width="22" height="16" rx="2" ry="2"></rect><line x1="1" y1="10" x2="23" y2="10"></line>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconCreditCard {
}
IconCreditCard.decorators = [
    { type: NgModule, args: [{
                declarations: [IconCreditCardComponent],
                exports: [IconCreditCardComponent]
            },] }
];
