// angular-feather/esm2015/feather/droplet.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconDropletComponent {
}
IconDropletComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-droplet',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-droplet">
    <path d="M12 2.69l5.66 5.66a8 8 0 1 1-11.31 0z"></path>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconDroplet {
}
IconDroplet.decorators = [
    { type: NgModule, args: [{
                declarations: [IconDropletComponent],
                exports: [IconDropletComponent]
            },] }
];
