// angular-feather/esm2015/feather/fast-forward.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconFastForwardComponent {
}
IconFastForwardComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-fast-forward',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-fast-forward">
    <polygon points="13 19 22 12 13 5 13 19"></polygon><polygon points="2 19 11 12 2 5 2 19"></polygon>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconFastForward {
}
IconFastForward.decorators = [
    { type: NgModule, args: [{
                declarations: [IconFastForwardComponent],
                exports: [IconFastForwardComponent]
            },] }
];
