// angular-feather/esm2015/feather/minus-square.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconMinusSquareComponent {
}
IconMinusSquareComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-minus-square',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-minus-square">
    <rect x="3" y="3" width="18" height="18" rx="2" ry="2"></rect><line x1="8" y1="12" x2="16" y2="12"></line>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconMinusSquare {
}
IconMinusSquare.decorators = [
    { type: NgModule, args: [{
                declarations: [IconMinusSquareComponent],
                exports: [IconMinusSquareComponent]
            },] }
];
