// angular-feather/esm2015/feather/minus-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconMinusCircleComponent {
}
IconMinusCircleComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-minus-circle',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-minus-circle">
    <circle cx="12" cy="12" r="10"></circle><line x1="8" y1="12" x2="16" y2="12"></line>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconMinusCircle {
}
IconMinusCircle.decorators = [
    { type: NgModule, args: [{
                declarations: [IconMinusCircleComponent],
                exports: [IconMinusCircleComponent]
            },] }
];
