// angular-feather/esm2015/feather/more-horizontal.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconMoreHorizontalComponent {
}
IconMoreHorizontalComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-more-horizontal',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-more-horizontal">
    <circle cx="12" cy="12" r="1"></circle><circle cx="19" cy="12" r="1"></circle><circle cx="5" cy="12" r="1"></circle>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconMoreHorizontal {
}
IconMoreHorizontal.decorators = [
    { type: NgModule, args: [{
                declarations: [IconMoreHorizontalComponent],
                exports: [IconMoreHorizontalComponent]
            },] }
];
