// angular-feather/esm2015/feather/more-vertical.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconMoreVerticalComponent {
}
IconMoreVerticalComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-more-vertical',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-more-vertical">
    <circle cx="12" cy="12" r="1"></circle><circle cx="12" cy="5" r="1"></circle><circle cx="12" cy="19" r="1"></circle>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconMoreVertical {
}
IconMoreVertical.decorators = [
    { type: NgModule, args: [{
                declarations: [IconMoreVerticalComponent],
                exports: [IconMoreVerticalComponent]
            },] }
];
