// angular-feather/esm2015/feather/navigation-2.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconNavigation2Component {
}
IconNavigation2Component.decorators = [
    { type: Component, args: [{
                selector: 'i-navigation-2',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-navigation-2">
    <polygon points="12 2 19 21 12 17 5 21 12 2"></polygon>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconNavigation2 {
}
IconNavigation2.decorators = [
    { type: NgModule, args: [{
                declarations: [IconNavigation2Component],
                exports: [IconNavigation2Component]
            },] }
];
