// angular-feather/esm2015/feather/navigation.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconNavigationComponent {
}
IconNavigationComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-navigation',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-navigation">
    <polygon points="3 11 22 2 13 21 11 13 3 11"></polygon>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconNavigation {
}
IconNavigation.decorators = [
    { type: NgModule, args: [{
                declarations: [IconNavigationComponent],
                exports: [IconNavigationComponent]
            },] }
];
