// angular-feather/esm2015/feather/rotate-ccw.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconRotateCcwComponent {
}
IconRotateCcwComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-rotate-ccw',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-rotate-ccw">
    <polyline points="1 4 1 10 7 10"></polyline><path d="M3.51 15a9 9 0 1 0 2.13-9.36L1 10"></path>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconRotateCcw {
}
IconRotateCcw.decorators = [
    { type: NgModule, args: [{
                declarations: [IconRotateCcwComponent],
                exports: [IconRotateCcwComponent]
            },] }
];
