// angular-feather/esm2015/feather/rotate-cw.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconRotateCwComponent {
}
IconRotateCwComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-rotate-cw',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-rotate-cw">
    <polyline points="23 4 23 10 17 10"></polyline><path d="M20.49 15a9 9 0 1 1-2.12-9.36L23 10"></path>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconRotateCw {
}
IconRotateCw.decorators = [
    { type: NgModule, args: [{
                declarations: [IconRotateCwComponent],
                exports: [IconRotateCwComponent]
            },] }
];
