// angular-feather/esm2015/feather/shield.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconShieldComponent {
}
IconShieldComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-shield',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-shield">
    <path d="M12 22s8-4 8-10V5l-8-3-8 3v7c0 6 8 10 8 10z"></path>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconShield {
}
IconShield.decorators = [
    { type: NgModule, args: [{
                declarations: [IconShieldComponent],
                exports: [IconShieldComponent]
            },] }
];
