// angular-feather/esm2015/feather/skip-forward.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconSkipForwardComponent {
}
IconSkipForwardComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-skip-forward',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-skip-forward">
    <polygon points="5 4 15 12 5 20 5 4"></polygon><line x1="19" y1="5" x2="19" y2="19"></line>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconSkipForward {
}
IconSkipForward.decorators = [
    { type: NgModule, args: [{
                declarations: [IconSkipForwardComponent],
                exports: [IconSkipForwardComponent]
            },] }
];
