// angular-feather/esm2015/feather/smartphone.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconSmartphoneComponent {
}
IconSmartphoneComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-smartphone',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-smartphone">
    <rect x="5" y="2" width="14" height="20" rx="2" ry="2"></rect><line x1="12" y1="18" x2="12" y2="18"></line>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconSmartphone {
}
IconSmartphone.decorators = [
    { type: NgModule, args: [{
                declarations: [IconSmartphoneComponent],
                exports: [IconSmartphoneComponent]
            },] }
];
