// angular-feather/esm2015/feather/stop-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconStopCircleComponent {
}
IconStopCircleComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-stop-circle',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-stop-circle">
    <circle cx="12" cy="12" r="10"></circle><rect x="9" y="9" width="6" height="6"></rect>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconStopCircle {
}
IconStopCircle.decorators = [
    { type: NgModule, args: [{
                declarations: [IconStopCircleComponent],
                exports: [IconStopCircleComponent]
            },] }
];
