// angular-feather/esm2015/feather/terminal.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconTerminalComponent {
}
IconTerminalComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-terminal',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-terminal">
    <polyline points="4 17 10 11 4 5"></polyline><line x1="12" y1="19" x2="20" y2="19"></line>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconTerminal {
}
IconTerminal.decorators = [
    { type: NgModule, args: [{
                declarations: [IconTerminalComponent],
                exports: [IconTerminalComponent]
            },] }
];
