// angular-feather/esm2015/feather/toggle-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconToggleLeftComponent {
}
IconToggleLeftComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-toggle-left',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-toggle-left">
    <rect x="1" y="5" width="22" height="14" rx="7" ry="7"></rect><circle cx="8" cy="12" r="3"></circle>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconToggleLeft {
}
IconToggleLeft.decorators = [
    { type: NgModule, args: [{
                declarations: [IconToggleLeftComponent],
                exports: [IconToggleLeftComponent]
            },] }
];
