// angular-feather/esm2015/feather/trending-up.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
export class IconTrendingUpComponent {
}
IconTrendingUpComponent.decorators = [
    { type: Component, args: [{
                selector: 'i-trending-up',
                template: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="feather feather-trending-up">
    <polyline points="23 6 13.5 15.5 8.5 10.5 1 18"></polyline><polyline points="17 6 23 6 23 12"></polyline>
  </svg>`,
                styles: [`
    :host {
      display: inline-block;
      width: 24px;
      height: 24px;
      fill: none;
      stroke: currentColor;
      stroke-width: 2px;
      stroke-linecap: round;
      stroke-linejoin: round;
    }
  `]
            }] }
];
export class IconTrendingUp {
}
IconTrendingUp.decorators = [
    { type: NgModule, args: [{
                declarations: [IconTrendingUpComponent],
                exports: [IconTrendingUpComponent]
            },] }
];
