// angular-feather/esm5/feather/activity.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconActivityComponent = /** @class */ (function () {
    function IconActivityComponent() {
    }
    IconActivityComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-activity',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-activity\">\n    <polyline points=\"22 12 18 12 15 21 9 3 6 12 2 12\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconActivityComponent;
}());
export { IconActivityComponent };
var IconActivity = /** @class */ (function () {
    function IconActivity() {
    }
    IconActivity.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconActivityComponent],
                    exports: [IconActivityComponent]
                },] }
    ];
    return IconActivity;
}());
export { IconActivity };
