// angular-feather/esm5/feather/airplay.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconAirplayComponent = /** @class */ (function () {
    function IconAirplayComponent() {
    }
    IconAirplayComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-airplay',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-airplay\">\n    <path d=\"M5 17H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h16a2 2 0 0 1 2 2v10a2 2 0 0 1-2 2h-1\"></path><polygon points=\"12 15 17 21 7 21 12 15\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconAirplayComponent;
}());
export { IconAirplayComponent };
var IconAirplay = /** @class */ (function () {
    function IconAirplay() {
    }
    IconAirplay.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconAirplayComponent],
                    exports: [IconAirplayComponent]
                },] }
    ];
    return IconAirplay;
}());
export { IconAirplay };
