// angular-feather/esm5/feather/alert-octagon.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconAlertOctagonComponent = /** @class */ (function () {
    function IconAlertOctagonComponent() {
    }
    IconAlertOctagonComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-alert-octagon',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-alert-octagon\">\n    <polygon points=\"7.86 2 16.14 2 22 7.86 22 16.14 16.14 22 7.86 22 2 16.14 2 7.86 7.86 2\"></polygon><line x1=\"12\" y1=\"8\" x2=\"12\" y2=\"12\"></line><line x1=\"12\" y1=\"16\" x2=\"12\" y2=\"16\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconAlertOctagonComponent;
}());
export { IconAlertOctagonComponent };
var IconAlertOctagon = /** @class */ (function () {
    function IconAlertOctagon() {
    }
    IconAlertOctagon.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconAlertOctagonComponent],
                    exports: [IconAlertOctagonComponent]
                },] }
    ];
    return IconAlertOctagon;
}());
export { IconAlertOctagon };
