// angular-feather/esm5/feather/alert-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconAlertCircleComponent = /** @class */ (function () {
    function IconAlertCircleComponent() {
    }
    IconAlertCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-alert-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-alert-circle\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><line x1=\"12\" y1=\"8\" x2=\"12\" y2=\"12\"></line><line x1=\"12\" y1=\"16\" x2=\"12\" y2=\"16\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconAlertCircleComponent;
}());
export { IconAlertCircleComponent };
var IconAlertCircle = /** @class */ (function () {
    function IconAlertCircle() {
    }
    IconAlertCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconAlertCircleComponent],
                    exports: [IconAlertCircleComponent]
                },] }
    ];
    return IconAlertCircle;
}());
export { IconAlertCircle };
