// angular-feather/esm5/feather/alert-triangle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconAlertTriangleComponent = /** @class */ (function () {
    function IconAlertTriangleComponent() {
    }
    IconAlertTriangleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-alert-triangle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-alert-triangle\">\n    <path d=\"M10.29 3.86L1.82 18a2 2 0 0 0 1.71 3h16.94a2 2 0 0 0 1.71-3L13.71 3.86a2 2 0 0 0-3.42 0z\"></path><line x1=\"12\" y1=\"9\" x2=\"12\" y2=\"13\"></line><line x1=\"12\" y1=\"17\" x2=\"12\" y2=\"17\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconAlertTriangleComponent;
}());
export { IconAlertTriangleComponent };
var IconAlertTriangle = /** @class */ (function () {
    function IconAlertTriangle() {
    }
    IconAlertTriangle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconAlertTriangleComponent],
                    exports: [IconAlertTriangleComponent]
                },] }
    ];
    return IconAlertTriangle;
}());
export { IconAlertTriangle };
