// angular-feather/esm5/feather/align-center.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconAlignCenterComponent = /** @class */ (function () {
    function IconAlignCenterComponent() {
    }
    IconAlignCenterComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-align-center',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-align-center\">\n    <line x1=\"18\" y1=\"10\" x2=\"6\" y2=\"10\"></line><line x1=\"21\" y1=\"6\" x2=\"3\" y2=\"6\"></line><line x1=\"21\" y1=\"14\" x2=\"3\" y2=\"14\"></line><line x1=\"18\" y1=\"18\" x2=\"6\" y2=\"18\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconAlignCenterComponent;
}());
export { IconAlignCenterComponent };
var IconAlignCenter = /** @class */ (function () {
    function IconAlignCenter() {
    }
    IconAlignCenter.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconAlignCenterComponent],
                    exports: [IconAlignCenterComponent]
                },] }
    ];
    return IconAlignCenter;
}());
export { IconAlignCenter };
