// angular-feather/esm5/feather/align-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconAlignRightComponent = /** @class */ (function () {
    function IconAlignRightComponent() {
    }
    IconAlignRightComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-align-right',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-align-right\">\n    <line x1=\"21\" y1=\"10\" x2=\"7\" y2=\"10\"></line><line x1=\"21\" y1=\"6\" x2=\"3\" y2=\"6\"></line><line x1=\"21\" y1=\"14\" x2=\"3\" y2=\"14\"></line><line x1=\"21\" y1=\"18\" x2=\"7\" y2=\"18\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconAlignRightComponent;
}());
export { IconAlignRightComponent };
var IconAlignRight = /** @class */ (function () {
    function IconAlignRight() {
    }
    IconAlignRight.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconAlignRightComponent],
                    exports: [IconAlignRightComponent]
                },] }
    ];
    return IconAlignRight;
}());
export { IconAlignRight };
