// angular-feather/esm5/feather/anchor.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconAnchorComponent = /** @class */ (function () {
    function IconAnchorComponent() {
    }
    IconAnchorComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-anchor',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-anchor\">\n    <circle cx=\"12\" cy=\"5\" r=\"3\"></circle><line x1=\"12\" y1=\"22\" x2=\"12\" y2=\"8\"></line><path d=\"M5 12H2a10 10 0 0 0 20 0h-3\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconAnchorComponent;
}());
export { IconAnchorComponent };
var IconAnchor = /** @class */ (function () {
    function IconAnchor() {
    }
    IconAnchor.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconAnchorComponent],
                    exports: [IconAnchorComponent]
                },] }
    ];
    return IconAnchor;
}());
export { IconAnchor };
