// angular-feather/esm5/feather/archive.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArchiveComponent = /** @class */ (function () {
    function IconArchiveComponent() {
    }
    IconArchiveComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-archive',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-archive\">\n    <polyline points=\"21 8 21 21 3 21 3 8\"></polyline><rect x=\"1\" y=\"3\" width=\"22\" height=\"5\"></rect><line x1=\"10\" y1=\"12\" x2=\"14\" y2=\"12\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArchiveComponent;
}());
export { IconArchiveComponent };
var IconArchive = /** @class */ (function () {
    function IconArchive() {
    }
    IconArchive.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArchiveComponent],
                    exports: [IconArchiveComponent]
                },] }
    ];
    return IconArchive;
}());
export { IconArchive };
