// angular-feather/esm5/feather/arrow-up.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArrowUpComponent = /** @class */ (function () {
    function IconArrowUpComponent() {
    }
    IconArrowUpComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-arrow-up',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-arrow-up\">\n    <line x1=\"12\" y1=\"19\" x2=\"12\" y2=\"5\"></line><polyline points=\"5 12 12 5 19 12\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArrowUpComponent;
}());
export { IconArrowUpComponent };
var IconArrowUp = /** @class */ (function () {
    function IconArrowUp() {
    }
    IconArrowUp.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArrowUpComponent],
                    exports: [IconArrowUpComponent]
                },] }
    ];
    return IconArrowUp;
}());
export { IconArrowUp };
