// angular-feather/esm5/feather/arrow-down-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArrowDownCircleComponent = /** @class */ (function () {
    function IconArrowDownCircleComponent() {
    }
    IconArrowDownCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-arrow-down-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-arrow-down-circle\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><polyline points=\"8 12 12 16 16 12\"></polyline><line x1=\"12\" y1=\"8\" x2=\"12\" y2=\"16\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArrowDownCircleComponent;
}());
export { IconArrowDownCircleComponent };
var IconArrowDownCircle = /** @class */ (function () {
    function IconArrowDownCircle() {
    }
    IconArrowDownCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArrowDownCircleComponent],
                    exports: [IconArrowDownCircleComponent]
                },] }
    ];
    return IconArrowDownCircle;
}());
export { IconArrowDownCircle };
