// angular-feather/esm5/feather/arrow-down.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArrowDownComponent = /** @class */ (function () {
    function IconArrowDownComponent() {
    }
    IconArrowDownComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-arrow-down',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-arrow-down\">\n    <line x1=\"12\" y1=\"5\" x2=\"12\" y2=\"19\"></line><polyline points=\"19 12 12 19 5 12\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArrowDownComponent;
}());
export { IconArrowDownComponent };
var IconArrowDown = /** @class */ (function () {
    function IconArrowDown() {
    }
    IconArrowDown.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArrowDownComponent],
                    exports: [IconArrowDownComponent]
                },] }
    ];
    return IconArrowDown;
}());
export { IconArrowDown };
