// angular-feather/esm5/feather/arrow-up-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArrowUpCircleComponent = /** @class */ (function () {
    function IconArrowUpCircleComponent() {
    }
    IconArrowUpCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-arrow-up-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-arrow-up-circle\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><polyline points=\"16 12 12 8 8 12\"></polyline><line x1=\"12\" y1=\"16\" x2=\"12\" y2=\"8\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArrowUpCircleComponent;
}());
export { IconArrowUpCircleComponent };
var IconArrowUpCircle = /** @class */ (function () {
    function IconArrowUpCircle() {
    }
    IconArrowUpCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArrowUpCircleComponent],
                    exports: [IconArrowUpCircleComponent]
                },] }
    ];
    return IconArrowUpCircle;
}());
export { IconArrowUpCircle };
