// angular-feather/esm5/feather/arrow-up-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArrowUpLeftComponent = /** @class */ (function () {
    function IconArrowUpLeftComponent() {
    }
    IconArrowUpLeftComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-arrow-up-left',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-arrow-up-left\">\n    <line x1=\"17\" y1=\"17\" x2=\"7\" y2=\"7\"></line><polyline points=\"7 17 7 7 17 7\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArrowUpLeftComponent;
}());
export { IconArrowUpLeftComponent };
var IconArrowUpLeft = /** @class */ (function () {
    function IconArrowUpLeft() {
    }
    IconArrowUpLeft.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArrowUpLeftComponent],
                    exports: [IconArrowUpLeftComponent]
                },] }
    ];
    return IconArrowUpLeft;
}());
export { IconArrowUpLeft };
