// angular-feather/esm5/feather/arrow-left-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArrowLeftCircleComponent = /** @class */ (function () {
    function IconArrowLeftCircleComponent() {
    }
    IconArrowLeftCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-arrow-left-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-arrow-left-circle\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><polyline points=\"12 8 8 12 12 16\"></polyline><line x1=\"16\" y1=\"12\" x2=\"8\" y2=\"12\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArrowLeftCircleComponent;
}());
export { IconArrowLeftCircleComponent };
var IconArrowLeftCircle = /** @class */ (function () {
    function IconArrowLeftCircle() {
    }
    IconArrowLeftCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArrowLeftCircleComponent],
                    exports: [IconArrowLeftCircleComponent]
                },] }
    ];
    return IconArrowLeftCircle;
}());
export { IconArrowLeftCircle };
