// angular-feather/esm5/feather/arrow-right-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArrowRightCircleComponent = /** @class */ (function () {
    function IconArrowRightCircleComponent() {
    }
    IconArrowRightCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-arrow-right-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-arrow-right-circle\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><polyline points=\"12 16 16 12 12 8\"></polyline><line x1=\"8\" y1=\"12\" x2=\"16\" y2=\"12\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArrowRightCircleComponent;
}());
export { IconArrowRightCircleComponent };
var IconArrowRightCircle = /** @class */ (function () {
    function IconArrowRightCircle() {
    }
    IconArrowRightCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArrowRightCircleComponent],
                    exports: [IconArrowRightCircleComponent]
                },] }
    ];
    return IconArrowRightCircle;
}());
export { IconArrowRightCircle };
