// angular-feather/esm5/feather/arrow-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArrowLeftComponent = /** @class */ (function () {
    function IconArrowLeftComponent() {
    }
    IconArrowLeftComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-arrow-left',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-arrow-left\">\n    <line x1=\"19\" y1=\"12\" x2=\"5\" y2=\"12\"></line><polyline points=\"12 19 5 12 12 5\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArrowLeftComponent;
}());
export { IconArrowLeftComponent };
var IconArrowLeft = /** @class */ (function () {
    function IconArrowLeft() {
    }
    IconArrowLeft.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArrowLeftComponent],
                    exports: [IconArrowLeftComponent]
                },] }
    ];
    return IconArrowLeft;
}());
export { IconArrowLeft };
