// angular-feather/esm5/feather/arrow-up-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArrowUpRightComponent = /** @class */ (function () {
    function IconArrowUpRightComponent() {
    }
    IconArrowUpRightComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-arrow-up-right',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-arrow-up-right\">\n    <line x1=\"7\" y1=\"17\" x2=\"17\" y2=\"7\"></line><polyline points=\"7 7 17 7 17 17\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArrowUpRightComponent;
}());
export { IconArrowUpRightComponent };
var IconArrowUpRight = /** @class */ (function () {
    function IconArrowUpRight() {
    }
    IconArrowUpRight.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArrowUpRightComponent],
                    exports: [IconArrowUpRightComponent]
                },] }
    ];
    return IconArrowUpRight;
}());
export { IconArrowUpRight };
