// angular-feather/esm5/feather/arrow-down-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArrowDownRightComponent = /** @class */ (function () {
    function IconArrowDownRightComponent() {
    }
    IconArrowDownRightComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-arrow-down-right',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-arrow-down-right\">\n    <line x1=\"7\" y1=\"7\" x2=\"17\" y2=\"17\"></line><polyline points=\"17 7 17 17 7 17\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArrowDownRightComponent;
}());
export { IconArrowDownRightComponent };
var IconArrowDownRight = /** @class */ (function () {
    function IconArrowDownRight() {
    }
    IconArrowDownRight.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArrowDownRightComponent],
                    exports: [IconArrowDownRightComponent]
                },] }
    ];
    return IconArrowDownRight;
}());
export { IconArrowDownRight };
