// angular-feather/esm5/feather/arrow-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArrowRightComponent = /** @class */ (function () {
    function IconArrowRightComponent() {
    }
    IconArrowRightComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-arrow-right',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-arrow-right\">\n    <line x1=\"5\" y1=\"12\" x2=\"19\" y2=\"12\"></line><polyline points=\"12 5 19 12 12 19\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArrowRightComponent;
}());
export { IconArrowRightComponent };
var IconArrowRight = /** @class */ (function () {
    function IconArrowRight() {
    }
    IconArrowRight.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArrowRightComponent],
                    exports: [IconArrowRightComponent]
                },] }
    ];
    return IconArrowRight;
}());
export { IconArrowRight };
