// angular-feather/esm5/feather/arrow-down-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconArrowDownLeftComponent = /** @class */ (function () {
    function IconArrowDownLeftComponent() {
    }
    IconArrowDownLeftComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-arrow-down-left',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-arrow-down-left\">\n    <line x1=\"17\" y1=\"7\" x2=\"7\" y2=\"17\"></line><polyline points=\"17 17 7 17 7 7\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconArrowDownLeftComponent;
}());
export { IconArrowDownLeftComponent };
var IconArrowDownLeft = /** @class */ (function () {
    function IconArrowDownLeft() {
    }
    IconArrowDownLeft.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconArrowDownLeftComponent],
                    exports: [IconArrowDownLeftComponent]
                },] }
    ];
    return IconArrowDownLeft;
}());
export { IconArrowDownLeft };
