// angular-feather/esm5/feather/at-sign.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconAtSignComponent = /** @class */ (function () {
    function IconAtSignComponent() {
    }
    IconAtSignComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-at-sign',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-at-sign\">\n    <circle cx=\"12\" cy=\"12\" r=\"4\"></circle><path d=\"M16 8v5a3 3 0 0 0 6 0v-1a10 10 0 1 0-3.92 7.94\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconAtSignComponent;
}());
export { IconAtSignComponent };
var IconAtSign = /** @class */ (function () {
    function IconAtSign() {
    }
    IconAtSign.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconAtSignComponent],
                    exports: [IconAtSignComponent]
                },] }
    ];
    return IconAtSign;
}());
export { IconAtSign };
