// angular-feather/esm5/feather/award.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconAwardComponent = /** @class */ (function () {
    function IconAwardComponent() {
    }
    IconAwardComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-award',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-award\">\n    <circle cx=\"12\" cy=\"8\" r=\"7\"></circle><polyline points=\"8.21 13.89 7 23 12 20 17 23 15.79 13.88\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconAwardComponent;
}());
export { IconAwardComponent };
var IconAward = /** @class */ (function () {
    function IconAward() {
    }
    IconAward.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconAwardComponent],
                    exports: [IconAwardComponent]
                },] }
    ];
    return IconAward;
}());
export { IconAward };
