// angular-feather/esm5/feather/bar-chart.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconBarChartComponent = /** @class */ (function () {
    function IconBarChartComponent() {
    }
    IconBarChartComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-bar-chart',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-bar-chart\">\n    <line x1=\"12\" y1=\"20\" x2=\"12\" y2=\"10\"></line><line x1=\"18\" y1=\"20\" x2=\"18\" y2=\"4\"></line><line x1=\"6\" y1=\"20\" x2=\"6\" y2=\"16\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconBarChartComponent;
}());
export { IconBarChartComponent };
var IconBarChart = /** @class */ (function () {
    function IconBarChart() {
    }
    IconBarChart.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconBarChartComponent],
                    exports: [IconBarChartComponent]
                },] }
    ];
    return IconBarChart;
}());
export { IconBarChart };
