// angular-feather/esm5/feather/battery-charging.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconBatteryChargingComponent = /** @class */ (function () {
    function IconBatteryChargingComponent() {
    }
    IconBatteryChargingComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-battery-charging',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-battery-charging\">\n    <path d=\"M5 18H3a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h3.19M15 6h2a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2h-3.19\"></path><line x1=\"23\" y1=\"13\" x2=\"23\" y2=\"11\"></line><polyline points=\"11 6 7 12 13 12 9 18\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconBatteryChargingComponent;
}());
export { IconBatteryChargingComponent };
var IconBatteryCharging = /** @class */ (function () {
    function IconBatteryCharging() {
    }
    IconBatteryCharging.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconBatteryChargingComponent],
                    exports: [IconBatteryChargingComponent]
                },] }
    ];
    return IconBatteryCharging;
}());
export { IconBatteryCharging };
