// angular-feather/esm5/feather/battery.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconBatteryComponent = /** @class */ (function () {
    function IconBatteryComponent() {
    }
    IconBatteryComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-battery',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-battery\">\n    <rect x=\"1\" y=\"6\" width=\"18\" height=\"12\" rx=\"2\" ry=\"2\"></rect><line x1=\"23\" y1=\"13\" x2=\"23\" y2=\"11\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconBatteryComponent;
}());
export { IconBatteryComponent };
var IconBattery = /** @class */ (function () {
    function IconBattery() {
    }
    IconBattery.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconBatteryComponent],
                    exports: [IconBatteryComponent]
                },] }
    ];
    return IconBattery;
}());
export { IconBattery };
