// angular-feather/esm5/feather/bell-off.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconBellOffComponent = /** @class */ (function () {
    function IconBellOffComponent() {
    }
    IconBellOffComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-bell-off',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-bell-off\">\n    <path d=\"M13.73 21a2 2 0 0 1-3.46 0\"></path><path d=\"M18.63 13A17.89 17.89 0 0 1 18 8\"></path><path d=\"M6.26 6.26A5.86 5.86 0 0 0 6 8c0 7-3 9-3 9h14\"></path><path d=\"M18 8a6 6 0 0 0-9.33-5\"></path><line x1=\"1\" y1=\"1\" x2=\"23\" y2=\"23\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconBellOffComponent;
}());
export { IconBellOffComponent };
var IconBellOff = /** @class */ (function () {
    function IconBellOff() {
    }
    IconBellOff.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconBellOffComponent],
                    exports: [IconBellOffComponent]
                },] }
    ];
    return IconBellOff;
}());
export { IconBellOff };
