// angular-feather/esm5/feather/bell.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconBellComponent = /** @class */ (function () {
    function IconBellComponent() {
    }
    IconBellComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-bell',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-bell\">\n    <path d=\"M18 8A6 6 0 0 0 6 8c0 7-3 9-3 9h18s-3-2-3-9\"></path><path d=\"M13.73 21a2 2 0 0 1-3.46 0\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconBellComponent;
}());
export { IconBellComponent };
var IconBell = /** @class */ (function () {
    function IconBell() {
    }
    IconBell.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconBellComponent],
                    exports: [IconBellComponent]
                },] }
    ];
    return IconBell;
}());
export { IconBell };
