// angular-feather/esm5/feather/bluetooth.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconBluetoothComponent = /** @class */ (function () {
    function IconBluetoothComponent() {
    }
    IconBluetoothComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-bluetooth',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-bluetooth\">\n    <polyline points=\"6.5 6.5 17.5 17.5 12 23 12 1 17.5 6.5 6.5 17.5\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconBluetoothComponent;
}());
export { IconBluetoothComponent };
var IconBluetooth = /** @class */ (function () {
    function IconBluetooth() {
    }
    IconBluetooth.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconBluetoothComponent],
                    exports: [IconBluetoothComponent]
                },] }
    ];
    return IconBluetooth;
}());
export { IconBluetooth };
