// angular-feather/esm5/feather/bold.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconBoldComponent = /** @class */ (function () {
    function IconBoldComponent() {
    }
    IconBoldComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-bold',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-bold\">\n    <path d=\"M6 4h8a4 4 0 0 1 4 4 4 4 0 0 1-4 4H6z\"></path><path d=\"M6 12h9a4 4 0 0 1 4 4 4 4 0 0 1-4 4H6z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconBoldComponent;
}());
export { IconBoldComponent };
var IconBold = /** @class */ (function () {
    function IconBold() {
    }
    IconBold.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconBoldComponent],
                    exports: [IconBoldComponent]
                },] }
    ];
    return IconBold;
}());
export { IconBold };
