// angular-feather/esm5/feather/book.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconBookComponent = /** @class */ (function () {
    function IconBookComponent() {
    }
    IconBookComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-book',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-book\">\n    <path d=\"M4 19.5A2.5 2.5 0 0 1 6.5 17H20\"></path><path d=\"M6.5 2H20v20H6.5A2.5 2.5 0 0 1 4 19.5v-15A2.5 2.5 0 0 1 6.5 2z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconBookComponent;
}());
export { IconBookComponent };
var IconBook = /** @class */ (function () {
    function IconBook() {
    }
    IconBook.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconBookComponent],
                    exports: [IconBookComponent]
                },] }
    ];
    return IconBook;
}());
export { IconBook };
