// angular-feather/esm5/feather/book-open.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconBookOpenComponent = /** @class */ (function () {
    function IconBookOpenComponent() {
    }
    IconBookOpenComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-book-open',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-book-open\">\n    <path d=\"M2 3h6a4 4 0 0 1 4 4v14a3 3 0 0 0-3-3H2z\"></path><path d=\"M22 3h-6a4 4 0 0 0-4 4v14a3 3 0 0 1 3-3h7z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconBookOpenComponent;
}());
export { IconBookOpenComponent };
var IconBookOpen = /** @class */ (function () {
    function IconBookOpen() {
    }
    IconBookOpen.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconBookOpenComponent],
                    exports: [IconBookOpenComponent]
                },] }
    ];
    return IconBookOpen;
}());
export { IconBookOpen };
