// angular-feather/esm5/feather/bookmark.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconBookmarkComponent = /** @class */ (function () {
    function IconBookmarkComponent() {
    }
    IconBookmarkComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-bookmark',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-bookmark\">\n    <path d=\"M19 21l-7-5-7 5V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconBookmarkComponent;
}());
export { IconBookmarkComponent };
var IconBookmark = /** @class */ (function () {
    function IconBookmark() {
    }
    IconBookmark.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconBookmarkComponent],
                    exports: [IconBookmarkComponent]
                },] }
    ];
    return IconBookmark;
}());
export { IconBookmark };
