// angular-feather/esm5/feather/box.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconBoxComponent = /** @class */ (function () {
    function IconBoxComponent() {
    }
    IconBoxComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-box',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-box\">\n    <path d=\"M12.89 1.45l8 4A2 2 0 0 1 22 7.24v9.53a2 2 0 0 1-1.11 1.79l-8 4a2 2 0 0 1-1.79 0l-8-4a2 2 0 0 1-1.1-1.8V7.24a2 2 0 0 1 1.11-1.79l8-4a2 2 0 0 1 1.78 0z\"></path><polyline points=\"2.32 6.16 12 11 21.68 6.16\"></polyline><line x1=\"12\" y1=\"22.76\" x2=\"12\" y2=\"11\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconBoxComponent;
}());
export { IconBoxComponent };
var IconBox = /** @class */ (function () {
    function IconBox() {
    }
    IconBox.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconBoxComponent],
                    exports: [IconBoxComponent]
                },] }
    ];
    return IconBox;
}());
export { IconBox };
