// angular-feather/esm5/feather/briefcase.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconBriefcaseComponent = /** @class */ (function () {
    function IconBriefcaseComponent() {
    }
    IconBriefcaseComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-briefcase',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-briefcase\">\n    <rect x=\"2\" y=\"7\" width=\"20\" height=\"14\" rx=\"2\" ry=\"2\"></rect><path d=\"M16 21V5a2 2 0 0 0-2-2h-4a2 2 0 0 0-2 2v16\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconBriefcaseComponent;
}());
export { IconBriefcaseComponent };
var IconBriefcase = /** @class */ (function () {
    function IconBriefcase() {
    }
    IconBriefcase.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconBriefcaseComponent],
                    exports: [IconBriefcaseComponent]
                },] }
    ];
    return IconBriefcase;
}());
export { IconBriefcase };
