// angular-feather/esm5/feather/camera.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCameraComponent = /** @class */ (function () {
    function IconCameraComponent() {
    }
    IconCameraComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-camera',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-camera\">\n    <path d=\"M23 19a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h4l2-3h6l2 3h4a2 2 0 0 1 2 2z\"></path><circle cx=\"12\" cy=\"13\" r=\"4\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCameraComponent;
}());
export { IconCameraComponent };
var IconCamera = /** @class */ (function () {
    function IconCamera() {
    }
    IconCamera.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCameraComponent],
                    exports: [IconCameraComponent]
                },] }
    ];
    return IconCamera;
}());
export { IconCamera };
