// angular-feather/esm5/feather/camera-off.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCameraOffComponent = /** @class */ (function () {
    function IconCameraOffComponent() {
    }
    IconCameraOffComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-camera-off',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-camera-off\">\n    <line x1=\"1\" y1=\"1\" x2=\"23\" y2=\"23\"></line><path d=\"M21 21H3a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h3m3-3h6l2 3h4a2 2 0 0 1 2 2v9.34m-7.72-2.06a4 4 0 1 1-5.56-5.56\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCameraOffComponent;
}());
export { IconCameraOffComponent };
var IconCameraOff = /** @class */ (function () {
    function IconCameraOff() {
    }
    IconCameraOff.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCameraOffComponent],
                    exports: [IconCameraOffComponent]
                },] }
    ];
    return IconCameraOff;
}());
export { IconCameraOff };
