// angular-feather/esm5/feather/cast.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCastComponent = /** @class */ (function () {
    function IconCastComponent() {
    }
    IconCastComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-cast',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-cast\">\n    <path d=\"M2 16.1A5 5 0 0 1 5.9 20M2 12.05A9 9 0 0 1 9.95 20M2 8V6a2 2 0 0 1 2-2h16a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2h-6\"></path><line x1=\"2\" y1=\"20\" x2=\"2\" y2=\"20\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCastComponent;
}());
export { IconCastComponent };
var IconCast = /** @class */ (function () {
    function IconCast() {
    }
    IconCast.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCastComponent],
                    exports: [IconCastComponent]
                },] }
    ];
    return IconCast;
}());
export { IconCast };
