// angular-feather/esm5/feather/check-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCheckCircleComponent = /** @class */ (function () {
    function IconCheckCircleComponent() {
    }
    IconCheckCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-check-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-check-circle\">\n    <path d=\"M22 11.08V12a10 10 0 1 1-5.93-9.14\"></path><polyline points=\"22 4 12 14.01 9 11.01\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCheckCircleComponent;
}());
export { IconCheckCircleComponent };
var IconCheckCircle = /** @class */ (function () {
    function IconCheckCircle() {
    }
    IconCheckCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCheckCircleComponent],
                    exports: [IconCheckCircleComponent]
                },] }
    ];
    return IconCheckCircle;
}());
export { IconCheckCircle };
