// angular-feather/esm5/feather/check.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCheckComponent = /** @class */ (function () {
    function IconCheckComponent() {
    }
    IconCheckComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-check',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-check\">\n    <polyline points=\"20 6 9 17 4 12\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCheckComponent;
}());
export { IconCheckComponent };
var IconCheck = /** @class */ (function () {
    function IconCheck() {
    }
    IconCheck.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCheckComponent],
                    exports: [IconCheckComponent]
                },] }
    ];
    return IconCheck;
}());
export { IconCheck };
