// angular-feather/esm5/feather/check-square.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCheckSquareComponent = /** @class */ (function () {
    function IconCheckSquareComponent() {
    }
    IconCheckSquareComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-check-square',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-check-square\">\n    <polyline points=\"9 11 12 14 22 4\"></polyline><path d=\"M21 12v7a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h11\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCheckSquareComponent;
}());
export { IconCheckSquareComponent };
var IconCheckSquare = /** @class */ (function () {
    function IconCheckSquare() {
    }
    IconCheckSquare.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCheckSquareComponent],
                    exports: [IconCheckSquareComponent]
                },] }
    ];
    return IconCheckSquare;
}());
export { IconCheckSquare };
