// angular-feather/esm5/feather/chevrons-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconChevronsLeftComponent = /** @class */ (function () {
    function IconChevronsLeftComponent() {
    }
    IconChevronsLeftComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-chevrons-left',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-chevrons-left\">\n    <polyline points=\"11 17 6 12 11 7\"></polyline><polyline points=\"18 17 13 12 18 7\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconChevronsLeftComponent;
}());
export { IconChevronsLeftComponent };
var IconChevronsLeft = /** @class */ (function () {
    function IconChevronsLeft() {
    }
    IconChevronsLeft.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconChevronsLeftComponent],
                    exports: [IconChevronsLeftComponent]
                },] }
    ];
    return IconChevronsLeft;
}());
export { IconChevronsLeft };
