// angular-feather/esm5/feather/chevron-up.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconChevronUpComponent = /** @class */ (function () {
    function IconChevronUpComponent() {
    }
    IconChevronUpComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-chevron-up',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-chevron-up\">\n    <polyline points=\"18 15 12 9 6 15\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconChevronUpComponent;
}());
export { IconChevronUpComponent };
var IconChevronUp = /** @class */ (function () {
    function IconChevronUp() {
    }
    IconChevronUp.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconChevronUpComponent],
                    exports: [IconChevronUpComponent]
                },] }
    ];
    return IconChevronUp;
}());
export { IconChevronUp };
