// angular-feather/esm5/feather/chevrons-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconChevronsRightComponent = /** @class */ (function () {
    function IconChevronsRightComponent() {
    }
    IconChevronsRightComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-chevrons-right',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-chevrons-right\">\n    <polyline points=\"13 17 18 12 13 7\"></polyline><polyline points=\"6 17 11 12 6 7\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconChevronsRightComponent;
}());
export { IconChevronsRightComponent };
var IconChevronsRight = /** @class */ (function () {
    function IconChevronsRight() {
    }
    IconChevronsRight.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconChevronsRightComponent],
                    exports: [IconChevronsRightComponent]
                },] }
    ];
    return IconChevronsRight;
}());
export { IconChevronsRight };
