// angular-feather/esm5/feather/chevrons-down.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconChevronsDownComponent = /** @class */ (function () {
    function IconChevronsDownComponent() {
    }
    IconChevronsDownComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-chevrons-down',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-chevrons-down\">\n    <polyline points=\"7 13 12 18 17 13\"></polyline><polyline points=\"7 6 12 11 17 6\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconChevronsDownComponent;
}());
export { IconChevronsDownComponent };
var IconChevronsDown = /** @class */ (function () {
    function IconChevronsDown() {
    }
    IconChevronsDown.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconChevronsDownComponent],
                    exports: [IconChevronsDownComponent]
                },] }
    ];
    return IconChevronsDown;
}());
export { IconChevronsDown };
