// angular-feather/esm5/feather/chevron-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconChevronRightComponent = /** @class */ (function () {
    function IconChevronRightComponent() {
    }
    IconChevronRightComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-chevron-right',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-chevron-right\">\n    <polyline points=\"9 18 15 12 9 6\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconChevronRightComponent;
}());
export { IconChevronRightComponent };
var IconChevronRight = /** @class */ (function () {
    function IconChevronRight() {
    }
    IconChevronRight.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconChevronRightComponent],
                    exports: [IconChevronRightComponent]
                },] }
    ];
    return IconChevronRight;
}());
export { IconChevronRight };
