// angular-feather/esm5/feather/chevrons-up.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconChevronsUpComponent = /** @class */ (function () {
    function IconChevronsUpComponent() {
    }
    IconChevronsUpComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-chevrons-up',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-chevrons-up\">\n    <polyline points=\"17 11 12 6 7 11\"></polyline><polyline points=\"17 18 12 13 7 18\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconChevronsUpComponent;
}());
export { IconChevronsUpComponent };
var IconChevronsUp = /** @class */ (function () {
    function IconChevronsUp() {
    }
    IconChevronsUp.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconChevronsUpComponent],
                    exports: [IconChevronsUpComponent]
                },] }
    ];
    return IconChevronsUp;
}());
export { IconChevronsUp };
