// angular-feather/esm5/feather/chevron-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconChevronLeftComponent = /** @class */ (function () {
    function IconChevronLeftComponent() {
    }
    IconChevronLeftComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-chevron-left',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-chevron-left\">\n    <polyline points=\"15 18 9 12 15 6\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconChevronLeftComponent;
}());
export { IconChevronLeftComponent };
var IconChevronLeft = /** @class */ (function () {
    function IconChevronLeft() {
    }
    IconChevronLeft.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconChevronLeftComponent],
                    exports: [IconChevronLeftComponent]
                },] }
    ];
    return IconChevronLeft;
}());
export { IconChevronLeft };
