// angular-feather/esm5/feather/chevron-down.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconChevronDownComponent = /** @class */ (function () {
    function IconChevronDownComponent() {
    }
    IconChevronDownComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-chevron-down',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-chevron-down\">\n    <polyline points=\"6 9 12 15 18 9\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconChevronDownComponent;
}());
export { IconChevronDownComponent };
var IconChevronDown = /** @class */ (function () {
    function IconChevronDown() {
    }
    IconChevronDown.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconChevronDownComponent],
                    exports: [IconChevronDownComponent]
                },] }
    ];
    return IconChevronDown;
}());
export { IconChevronDown };
