// angular-feather/esm5/feather/chrome.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconChromeComponent = /** @class */ (function () {
    function IconChromeComponent() {
    }
    IconChromeComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-chrome',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-chrome\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><circle cx=\"12\" cy=\"12\" r=\"4\"></circle><line x1=\"21.17\" y1=\"8\" x2=\"12\" y2=\"8\"></line><line x1=\"3.95\" y1=\"6.06\" x2=\"8.54\" y2=\"14\"></line><line x1=\"10.88\" y1=\"21.94\" x2=\"15.46\" y2=\"14\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconChromeComponent;
}());
export { IconChromeComponent };
var IconChrome = /** @class */ (function () {
    function IconChrome() {
    }
    IconChrome.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconChromeComponent],
                    exports: [IconChromeComponent]
                },] }
    ];
    return IconChrome;
}());
export { IconChrome };
