// angular-feather/esm5/feather/circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCircleComponent = /** @class */ (function () {
    function IconCircleComponent() {
    }
    IconCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-circle\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCircleComponent;
}());
export { IconCircleComponent };
var IconCircle = /** @class */ (function () {
    function IconCircle() {
    }
    IconCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCircleComponent],
                    exports: [IconCircleComponent]
                },] }
    ];
    return IconCircle;
}());
export { IconCircle };
