// angular-feather/esm5/feather/clipboard.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconClipboardComponent = /** @class */ (function () {
    function IconClipboardComponent() {
    }
    IconClipboardComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-clipboard',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-clipboard\">\n    <path d=\"M16 4h2a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2V6a2 2 0 0 1 2-2h2\"></path><rect x=\"8\" y=\"2\" width=\"8\" height=\"4\" rx=\"1\" ry=\"1\"></rect>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconClipboardComponent;
}());
export { IconClipboardComponent };
var IconClipboard = /** @class */ (function () {
    function IconClipboard() {
    }
    IconClipboard.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconClipboardComponent],
                    exports: [IconClipboardComponent]
                },] }
    ];
    return IconClipboard;
}());
export { IconClipboard };
