// angular-feather/esm5/feather/clock.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconClockComponent = /** @class */ (function () {
    function IconClockComponent() {
    }
    IconClockComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-clock',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-clock\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><polyline points=\"12 6 12 12 16 14\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconClockComponent;
}());
export { IconClockComponent };
var IconClock = /** @class */ (function () {
    function IconClock() {
    }
    IconClock.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconClockComponent],
                    exports: [IconClockComponent]
                },] }
    ];
    return IconClock;
}());
export { IconClock };
