// angular-feather/esm5/feather/cloud.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCloudComponent = /** @class */ (function () {
    function IconCloudComponent() {
    }
    IconCloudComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-cloud',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-cloud\">\n    <path d=\"M18 10h-1.26A8 8 0 1 0 9 20h9a5 5 0 0 0 0-10z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCloudComponent;
}());
export { IconCloudComponent };
var IconCloud = /** @class */ (function () {
    function IconCloud() {
    }
    IconCloud.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCloudComponent],
                    exports: [IconCloudComponent]
                },] }
    ];
    return IconCloud;
}());
export { IconCloud };
