// angular-feather/esm5/feather/cloud-drizzle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCloudDrizzleComponent = /** @class */ (function () {
    function IconCloudDrizzleComponent() {
    }
    IconCloudDrizzleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-cloud-drizzle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-cloud-drizzle\">\n    <line x1=\"8\" y1=\"19\" x2=\"8\" y2=\"21\"></line><line x1=\"8\" y1=\"13\" x2=\"8\" y2=\"15\"></line><line x1=\"16\" y1=\"19\" x2=\"16\" y2=\"21\"></line><line x1=\"16\" y1=\"13\" x2=\"16\" y2=\"15\"></line><line x1=\"12\" y1=\"21\" x2=\"12\" y2=\"23\"></line><line x1=\"12\" y1=\"15\" x2=\"12\" y2=\"17\"></line><path d=\"M20 16.58A5 5 0 0 0 18 7h-1.26A8 8 0 1 0 4 15.25\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCloudDrizzleComponent;
}());
export { IconCloudDrizzleComponent };
var IconCloudDrizzle = /** @class */ (function () {
    function IconCloudDrizzle() {
    }
    IconCloudDrizzle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCloudDrizzleComponent],
                    exports: [IconCloudDrizzleComponent]
                },] }
    ];
    return IconCloudDrizzle;
}());
export { IconCloudDrizzle };
