// angular-feather/esm5/feather/cloud-lightning.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCloudLightningComponent = /** @class */ (function () {
    function IconCloudLightningComponent() {
    }
    IconCloudLightningComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-cloud-lightning',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-cloud-lightning\">\n    <path d=\"M19 16.9A5 5 0 0 0 18 7h-1.26a8 8 0 1 0-11.62 9\"></path><polyline points=\"13 11 9 17 15 17 11 23\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCloudLightningComponent;
}());
export { IconCloudLightningComponent };
var IconCloudLightning = /** @class */ (function () {
    function IconCloudLightning() {
    }
    IconCloudLightning.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCloudLightningComponent],
                    exports: [IconCloudLightningComponent]
                },] }
    ];
    return IconCloudLightning;
}());
export { IconCloudLightning };
