// angular-feather/esm5/feather/cloud-off.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCloudOffComponent = /** @class */ (function () {
    function IconCloudOffComponent() {
    }
    IconCloudOffComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-cloud-off',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-cloud-off\">\n    <path d=\"M22.61 16.95A5 5 0 0 0 18 10h-1.26a8 8 0 0 0-7.05-6M5 5a8 8 0 0 0 4 15h9a5 5 0 0 0 1.7-.3\"></path><line x1=\"1\" y1=\"1\" x2=\"23\" y2=\"23\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCloudOffComponent;
}());
export { IconCloudOffComponent };
var IconCloudOff = /** @class */ (function () {
    function IconCloudOff() {
    }
    IconCloudOff.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCloudOffComponent],
                    exports: [IconCloudOffComponent]
                },] }
    ];
    return IconCloudOff;
}());
export { IconCloudOff };
