// angular-feather/esm5/feather/cloud-snow.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCloudSnowComponent = /** @class */ (function () {
    function IconCloudSnowComponent() {
    }
    IconCloudSnowComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-cloud-snow',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-cloud-snow\">\n    <path d=\"M20 17.58A5 5 0 0 0 18 8h-1.26A8 8 0 1 0 4 16.25\"></path><line x1=\"8\" y1=\"16\" x2=\"8\" y2=\"16\"></line><line x1=\"8\" y1=\"20\" x2=\"8\" y2=\"20\"></line><line x1=\"12\" y1=\"18\" x2=\"12\" y2=\"18\"></line><line x1=\"12\" y1=\"22\" x2=\"12\" y2=\"22\"></line><line x1=\"16\" y1=\"16\" x2=\"16\" y2=\"16\"></line><line x1=\"16\" y1=\"20\" x2=\"16\" y2=\"20\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCloudSnowComponent;
}());
export { IconCloudSnowComponent };
var IconCloudSnow = /** @class */ (function () {
    function IconCloudSnow() {
    }
    IconCloudSnow.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCloudSnowComponent],
                    exports: [IconCloudSnowComponent]
                },] }
    ];
    return IconCloudSnow;
}());
export { IconCloudSnow };
