// angular-feather/esm5/feather/codepen.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCodepenComponent = /** @class */ (function () {
    function IconCodepenComponent() {
    }
    IconCodepenComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-codepen',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-codepen\">\n    <polygon points=\"12 2 22 8.5 22 15.5 12 22 2 15.5 2 8.5 12 2\"></polygon><line x1=\"12\" y1=\"22\" x2=\"12\" y2=\"15.5\"></line><polyline points=\"22 8.5 12 15.5 2 8.5\"></polyline><polyline points=\"2 15.5 12 8.5 22 15.5\"></polyline><line x1=\"12\" y1=\"2\" x2=\"12\" y2=\"8.5\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCodepenComponent;
}());
export { IconCodepenComponent };
var IconCodepen = /** @class */ (function () {
    function IconCodepen() {
    }
    IconCodepen.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCodepenComponent],
                    exports: [IconCodepenComponent]
                },] }
    ];
    return IconCodepen;
}());
export { IconCodepen };
