// angular-feather/esm5/feather/code.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCodeComponent = /** @class */ (function () {
    function IconCodeComponent() {
    }
    IconCodeComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-code',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-code\">\n    <polyline points=\"16 18 22 12 16 6\"></polyline><polyline points=\"8 6 2 12 8 18\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCodeComponent;
}());
export { IconCodeComponent };
var IconCode = /** @class */ (function () {
    function IconCode() {
    }
    IconCode.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCodeComponent],
                    exports: [IconCodeComponent]
                },] }
    ];
    return IconCode;
}());
export { IconCode };
