// angular-feather/esm5/feather/coffee.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCoffeeComponent = /** @class */ (function () {
    function IconCoffeeComponent() {
    }
    IconCoffeeComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-coffee',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-coffee\">\n    <path d=\"M18 8h1a4 4 0 0 1 0 8h-1\"></path><path d=\"M2 8h16v9a4 4 0 0 1-4 4H6a4 4 0 0 1-4-4V8z\"></path><line x1=\"6\" y1=\"1\" x2=\"6\" y2=\"4\"></line><line x1=\"10\" y1=\"1\" x2=\"10\" y2=\"4\"></line><line x1=\"14\" y1=\"1\" x2=\"14\" y2=\"4\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCoffeeComponent;
}());
export { IconCoffeeComponent };
var IconCoffee = /** @class */ (function () {
    function IconCoffee() {
    }
    IconCoffee.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCoffeeComponent],
                    exports: [IconCoffeeComponent]
                },] }
    ];
    return IconCoffee;
}());
export { IconCoffee };
