// angular-feather/esm5/feather/command.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCommandComponent = /** @class */ (function () {
    function IconCommandComponent() {
    }
    IconCommandComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-command',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-command\">\n    <path d=\"M18 3a3 3 0 0 0-3 3v12a3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3H6a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3V6a3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3h12a3 3 0 0 0 3-3 3 3 0 0 0-3-3z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCommandComponent;
}());
export { IconCommandComponent };
var IconCommand = /** @class */ (function () {
    function IconCommand() {
    }
    IconCommand.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCommandComponent],
                    exports: [IconCommandComponent]
                },] }
    ];
    return IconCommand;
}());
export { IconCommand };
