// angular-feather/esm5/feather/compass.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCompassComponent = /** @class */ (function () {
    function IconCompassComponent() {
    }
    IconCompassComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-compass',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-compass\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><polygon points=\"16.24 7.76 14.12 14.12 7.76 16.24 9.88 9.88 16.24 7.76\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCompassComponent;
}());
export { IconCompassComponent };
var IconCompass = /** @class */ (function () {
    function IconCompass() {
    }
    IconCompass.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCompassComponent],
                    exports: [IconCompassComponent]
                },] }
    ];
    return IconCompass;
}());
export { IconCompass };
