// angular-feather/esm5/feather/copy.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCopyComponent = /** @class */ (function () {
    function IconCopyComponent() {
    }
    IconCopyComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-copy',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-copy\">\n    <rect x=\"9\" y=\"9\" width=\"13\" height=\"13\" rx=\"2\" ry=\"2\"></rect><path d=\"M5 15H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2h9a2 2 0 0 1 2 2v1\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCopyComponent;
}());
export { IconCopyComponent };
var IconCopy = /** @class */ (function () {
    function IconCopy() {
    }
    IconCopy.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCopyComponent],
                    exports: [IconCopyComponent]
                },] }
    ];
    return IconCopy;
}());
export { IconCopy };
