// angular-feather/esm5/feather/corner-left-up.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCornerLeftUpComponent = /** @class */ (function () {
    function IconCornerLeftUpComponent() {
    }
    IconCornerLeftUpComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-corner-left-up',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-corner-left-up\">\n    <polyline points=\"14 9 9 4 4 9\"></polyline><path d=\"M20 20h-7a4 4 0 0 1-4-4V4\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCornerLeftUpComponent;
}());
export { IconCornerLeftUpComponent };
var IconCornerLeftUp = /** @class */ (function () {
    function IconCornerLeftUp() {
    }
    IconCornerLeftUp.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCornerLeftUpComponent],
                    exports: [IconCornerLeftUpComponent]
                },] }
    ];
    return IconCornerLeftUp;
}());
export { IconCornerLeftUp };
