// angular-feather/esm5/feather/corner-right-up.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCornerRightUpComponent = /** @class */ (function () {
    function IconCornerRightUpComponent() {
    }
    IconCornerRightUpComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-corner-right-up',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-corner-right-up\">\n    <polyline points=\"10 9 15 4 20 9\"></polyline><path d=\"M4 20h7a4 4 0 0 0 4-4V4\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCornerRightUpComponent;
}());
export { IconCornerRightUpComponent };
var IconCornerRightUp = /** @class */ (function () {
    function IconCornerRightUp() {
    }
    IconCornerRightUp.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCornerRightUpComponent],
                    exports: [IconCornerRightUpComponent]
                },] }
    ];
    return IconCornerRightUp;
}());
export { IconCornerRightUp };
