// angular-feather/esm5/feather/corner-up-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCornerUpRightComponent = /** @class */ (function () {
    function IconCornerUpRightComponent() {
    }
    IconCornerUpRightComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-corner-up-right',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-corner-up-right\">\n    <polyline points=\"15 14 20 9 15 4\"></polyline><path d=\"M4 20v-7a4 4 0 0 1 4-4h12\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCornerUpRightComponent;
}());
export { IconCornerUpRightComponent };
var IconCornerUpRight = /** @class */ (function () {
    function IconCornerUpRight() {
    }
    IconCornerUpRight.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCornerUpRightComponent],
                    exports: [IconCornerUpRightComponent]
                },] }
    ];
    return IconCornerUpRight;
}());
export { IconCornerUpRight };
