// angular-feather/esm5/feather/corner-right-down.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCornerRightDownComponent = /** @class */ (function () {
    function IconCornerRightDownComponent() {
    }
    IconCornerRightDownComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-corner-right-down',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-corner-right-down\">\n    <polyline points=\"10 15 15 20 20 15\"></polyline><path d=\"M4 4h7a4 4 0 0 1 4 4v12\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCornerRightDownComponent;
}());
export { IconCornerRightDownComponent };
var IconCornerRightDown = /** @class */ (function () {
    function IconCornerRightDown() {
    }
    IconCornerRightDown.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCornerRightDownComponent],
                    exports: [IconCornerRightDownComponent]
                },] }
    ];
    return IconCornerRightDown;
}());
export { IconCornerRightDown };
