// angular-feather/esm5/feather/corner-down-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCornerDownRightComponent = /** @class */ (function () {
    function IconCornerDownRightComponent() {
    }
    IconCornerDownRightComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-corner-down-right',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-corner-down-right\">\n    <polyline points=\"15 10 20 15 15 20\"></polyline><path d=\"M4 4v7a4 4 0 0 0 4 4h12\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCornerDownRightComponent;
}());
export { IconCornerDownRightComponent };
var IconCornerDownRight = /** @class */ (function () {
    function IconCornerDownRight() {
    }
    IconCornerDownRight.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCornerDownRightComponent],
                    exports: [IconCornerDownRightComponent]
                },] }
    ];
    return IconCornerDownRight;
}());
export { IconCornerDownRight };
