// angular-feather/esm5/feather/corner-up-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCornerUpLeftComponent = /** @class */ (function () {
    function IconCornerUpLeftComponent() {
    }
    IconCornerUpLeftComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-corner-up-left',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-corner-up-left\">\n    <polyline points=\"9 14 4 9 9 4\"></polyline><path d=\"M20 20v-7a4 4 0 0 0-4-4H4\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCornerUpLeftComponent;
}());
export { IconCornerUpLeftComponent };
var IconCornerUpLeft = /** @class */ (function () {
    function IconCornerUpLeft() {
    }
    IconCornerUpLeft.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCornerUpLeftComponent],
                    exports: [IconCornerUpLeftComponent]
                },] }
    ];
    return IconCornerUpLeft;
}());
export { IconCornerUpLeft };
