// angular-feather/esm5/feather/corner-down-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCornerDownLeftComponent = /** @class */ (function () {
    function IconCornerDownLeftComponent() {
    }
    IconCornerDownLeftComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-corner-down-left',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-corner-down-left\">\n    <polyline points=\"9 10 4 15 9 20\"></polyline><path d=\"M20 4v7a4 4 0 0 1-4 4H4\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCornerDownLeftComponent;
}());
export { IconCornerDownLeftComponent };
var IconCornerDownLeft = /** @class */ (function () {
    function IconCornerDownLeft() {
    }
    IconCornerDownLeft.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCornerDownLeftComponent],
                    exports: [IconCornerDownLeftComponent]
                },] }
    ];
    return IconCornerDownLeft;
}());
export { IconCornerDownLeft };
