// angular-feather/esm5/feather/corner-left-down.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCornerLeftDownComponent = /** @class */ (function () {
    function IconCornerLeftDownComponent() {
    }
    IconCornerLeftDownComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-corner-left-down',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-corner-left-down\">\n    <polyline points=\"14 15 9 20 4 15\"></polyline><path d=\"M20 4h-7a4 4 0 0 0-4 4v12\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCornerLeftDownComponent;
}());
export { IconCornerLeftDownComponent };
var IconCornerLeftDown = /** @class */ (function () {
    function IconCornerLeftDown() {
    }
    IconCornerLeftDown.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCornerLeftDownComponent],
                    exports: [IconCornerLeftDownComponent]
                },] }
    ];
    return IconCornerLeftDown;
}());
export { IconCornerLeftDown };
