// angular-feather/esm5/feather/credit-card.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCreditCardComponent = /** @class */ (function () {
    function IconCreditCardComponent() {
    }
    IconCreditCardComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-credit-card',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-credit-card\">\n    <rect x=\"1\" y=\"4\" width=\"22\" height=\"16\" rx=\"2\" ry=\"2\"></rect><line x1=\"1\" y1=\"10\" x2=\"23\" y2=\"10\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCreditCardComponent;
}());
export { IconCreditCardComponent };
var IconCreditCard = /** @class */ (function () {
    function IconCreditCard() {
    }
    IconCreditCard.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCreditCardComponent],
                    exports: [IconCreditCardComponent]
                },] }
    ];
    return IconCreditCard;
}());
export { IconCreditCard };
