// angular-feather/esm5/feather/crosshair.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconCrosshairComponent = /** @class */ (function () {
    function IconCrosshairComponent() {
    }
    IconCrosshairComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-crosshair',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-crosshair\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><line x1=\"22\" y1=\"12\" x2=\"18\" y2=\"12\"></line><line x1=\"6\" y1=\"12\" x2=\"2\" y2=\"12\"></line><line x1=\"12\" y1=\"6\" x2=\"12\" y2=\"2\"></line><line x1=\"12\" y1=\"22\" x2=\"12\" y2=\"18\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconCrosshairComponent;
}());
export { IconCrosshairComponent };
var IconCrosshair = /** @class */ (function () {
    function IconCrosshair() {
    }
    IconCrosshair.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconCrosshairComponent],
                    exports: [IconCrosshairComponent]
                },] }
    ];
    return IconCrosshair;
}());
export { IconCrosshair };
