// angular-feather/esm5/feather/delete.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconDeleteComponent = /** @class */ (function () {
    function IconDeleteComponent() {
    }
    IconDeleteComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-delete',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-delete\">\n    <path d=\"M21 4H8l-7 8 7 8h13a2 2 0 0 0 2-2V6a2 2 0 0 0-2-2z\"></path><line x1=\"18\" y1=\"9\" x2=\"12\" y2=\"15\"></line><line x1=\"12\" y1=\"9\" x2=\"18\" y2=\"15\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconDeleteComponent;
}());
export { IconDeleteComponent };
var IconDelete = /** @class */ (function () {
    function IconDelete() {
    }
    IconDelete.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconDeleteComponent],
                    exports: [IconDeleteComponent]
                },] }
    ];
    return IconDelete;
}());
export { IconDelete };
