// angular-feather/esm5/feather/disc.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconDiscComponent = /** @class */ (function () {
    function IconDiscComponent() {
    }
    IconDiscComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-disc',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-disc\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><circle cx=\"12\" cy=\"12\" r=\"3\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconDiscComponent;
}());
export { IconDiscComponent };
var IconDisc = /** @class */ (function () {
    function IconDisc() {
    }
    IconDisc.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconDiscComponent],
                    exports: [IconDiscComponent]
                },] }
    ];
    return IconDisc;
}());
export { IconDisc };
