// angular-feather/esm5/feather/dollar-sign.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconDollarSignComponent = /** @class */ (function () {
    function IconDollarSignComponent() {
    }
    IconDollarSignComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-dollar-sign',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-dollar-sign\">\n    <line x1=\"12\" y1=\"1\" x2=\"12\" y2=\"23\"></line><path d=\"M17 5H9.5a3.5 3.5 0 0 0 0 7h5a3.5 3.5 0 0 1 0 7H6\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconDollarSignComponent;
}());
export { IconDollarSignComponent };
var IconDollarSign = /** @class */ (function () {
    function IconDollarSign() {
    }
    IconDollarSign.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconDollarSignComponent],
                    exports: [IconDollarSignComponent]
                },] }
    ];
    return IconDollarSign;
}());
export { IconDollarSign };
