// angular-feather/esm5/feather/download.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconDownloadComponent = /** @class */ (function () {
    function IconDownloadComponent() {
    }
    IconDownloadComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-download',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-download\">\n    <path d=\"M21 15v4a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-4\"></path><polyline points=\"7 10 12 15 17 10\"></polyline><line x1=\"12\" y1=\"15\" x2=\"12\" y2=\"3\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconDownloadComponent;
}());
export { IconDownloadComponent };
var IconDownload = /** @class */ (function () {
    function IconDownload() {
    }
    IconDownload.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconDownloadComponent],
                    exports: [IconDownloadComponent]
                },] }
    ];
    return IconDownload;
}());
export { IconDownload };
