// angular-feather/esm5/feather/download-cloud.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconDownloadCloudComponent = /** @class */ (function () {
    function IconDownloadCloudComponent() {
    }
    IconDownloadCloudComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-download-cloud',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-download-cloud\">\n    <polyline points=\"8 17 12 21 16 17\"></polyline><line x1=\"12\" y1=\"12\" x2=\"12\" y2=\"21\"></line><path d=\"M20.88 18.09A5 5 0 0 0 18 9h-1.26A8 8 0 1 0 3 16.29\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconDownloadCloudComponent;
}());
export { IconDownloadCloudComponent };
var IconDownloadCloud = /** @class */ (function () {
    function IconDownloadCloud() {
    }
    IconDownloadCloud.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconDownloadCloudComponent],
                    exports: [IconDownloadCloudComponent]
                },] }
    ];
    return IconDownloadCloud;
}());
export { IconDownloadCloud };
