// angular-feather/esm5/feather/edit-3.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconEdit3Component = /** @class */ (function () {
    function IconEdit3Component() {
    }
    IconEdit3Component.decorators = [
        { type: Component, args: [{
                    selector: 'i-edit-3',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-edit-3\">\n    <path d=\"M12 20h9\"></path><path d=\"M16.5 3.5a2.121 2.121 0 0 1 3 3L7 19l-4 1 1-4L16.5 3.5z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconEdit3Component;
}());
export { IconEdit3Component };
var IconEdit3 = /** @class */ (function () {
    function IconEdit3() {
    }
    IconEdit3.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconEdit3Component],
                    exports: [IconEdit3Component]
                },] }
    ];
    return IconEdit3;
}());
export { IconEdit3 };
