// angular-feather/esm5/feather/edit.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconEditComponent = /** @class */ (function () {
    function IconEditComponent() {
    }
    IconEditComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-edit',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-edit\">\n    <path d=\"M11 4H4a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14a2 2 0 0 0 2-2v-7\"></path><path d=\"M18.5 2.5a2.121 2.121 0 0 1 3 3L12 15l-4 1 1-4 9.5-9.5z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconEditComponent;
}());
export { IconEditComponent };
var IconEdit = /** @class */ (function () {
    function IconEdit() {
    }
    IconEdit.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconEditComponent],
                    exports: [IconEditComponent]
                },] }
    ];
    return IconEdit;
}());
export { IconEdit };
