// angular-feather/esm5/feather/edit-2.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconEdit2Component = /** @class */ (function () {
    function IconEdit2Component() {
    }
    IconEdit2Component.decorators = [
        { type: Component, args: [{
                    selector: 'i-edit-2',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-edit-2\">\n    <path d=\"M17 3a2.828 2.828 0 1 1 4 4L7.5 20.5 2 22l1.5-5.5L17 3z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconEdit2Component;
}());
export { IconEdit2Component };
var IconEdit2 = /** @class */ (function () {
    function IconEdit2() {
    }
    IconEdit2.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconEdit2Component],
                    exports: [IconEdit2Component]
                },] }
    ];
    return IconEdit2;
}());
export { IconEdit2 };
