// angular-feather/esm5/feather/external-link.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconExternalLinkComponent = /** @class */ (function () {
    function IconExternalLinkComponent() {
    }
    IconExternalLinkComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-external-link',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-external-link\">\n    <path d=\"M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6\"></path><polyline points=\"15 3 21 3 21 9\"></polyline><line x1=\"10\" y1=\"14\" x2=\"21\" y2=\"3\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconExternalLinkComponent;
}());
export { IconExternalLinkComponent };
var IconExternalLink = /** @class */ (function () {
    function IconExternalLink() {
    }
    IconExternalLink.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconExternalLinkComponent],
                    exports: [IconExternalLinkComponent]
                },] }
    ];
    return IconExternalLink;
}());
export { IconExternalLink };
