// angular-feather/esm5/feather/eye-off.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconEyeOffComponent = /** @class */ (function () {
    function IconEyeOffComponent() {
    }
    IconEyeOffComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-eye-off',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-eye-off\">\n    <path d=\"M17.94 17.94A10.07 10.07 0 0 1 12 20c-7 0-11-8-11-8a18.45 18.45 0 0 1 5.06-5.94M9.9 4.24A9.12 9.12 0 0 1 12 4c7 0 11 8 11 8a18.5 18.5 0 0 1-2.16 3.19m-6.72-1.07a3 3 0 1 1-4.24-4.24\"></path><line x1=\"1\" y1=\"1\" x2=\"23\" y2=\"23\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconEyeOffComponent;
}());
export { IconEyeOffComponent };
var IconEyeOff = /** @class */ (function () {
    function IconEyeOff() {
    }
    IconEyeOff.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconEyeOffComponent],
                    exports: [IconEyeOffComponent]
                },] }
    ];
    return IconEyeOff;
}());
export { IconEyeOff };
