// angular-feather/esm5/feather/eye.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconEyeComponent = /** @class */ (function () {
    function IconEyeComponent() {
    }
    IconEyeComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-eye',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-eye\">\n    <path d=\"M1 12s4-8 11-8 11 8 11 8-4 8-11 8-11-8-11-8z\"></path><circle cx=\"12\" cy=\"12\" r=\"3\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconEyeComponent;
}());
export { IconEyeComponent };
var IconEye = /** @class */ (function () {
    function IconEye() {
    }
    IconEye.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconEyeComponent],
                    exports: [IconEyeComponent]
                },] }
    ];
    return IconEye;
}());
export { IconEye };
