// angular-feather/esm5/feather/facebook.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconFacebookComponent = /** @class */ (function () {
    function IconFacebookComponent() {
    }
    IconFacebookComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-facebook',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-facebook\">\n    <path d=\"M18 2h-3a5 5 0 0 0-5 5v3H7v4h3v8h4v-8h3l1-4h-4V7a1 1 0 0 1 1-1h3z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconFacebookComponent;
}());
export { IconFacebookComponent };
var IconFacebook = /** @class */ (function () {
    function IconFacebook() {
    }
    IconFacebook.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconFacebookComponent],
                    exports: [IconFacebookComponent]
                },] }
    ];
    return IconFacebook;
}());
export { IconFacebook };
