// angular-feather/esm5/feather/fast-forward.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconFastForwardComponent = /** @class */ (function () {
    function IconFastForwardComponent() {
    }
    IconFastForwardComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-fast-forward',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-fast-forward\">\n    <polygon points=\"13 19 22 12 13 5 13 19\"></polygon><polygon points=\"2 19 11 12 2 5 2 19\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconFastForwardComponent;
}());
export { IconFastForwardComponent };
var IconFastForward = /** @class */ (function () {
    function IconFastForward() {
    }
    IconFastForward.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconFastForwardComponent],
                    exports: [IconFastForwardComponent]
                },] }
    ];
    return IconFastForward;
}());
export { IconFastForward };
