// angular-feather/esm5/feather/feather.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconFeatherComponent = /** @class */ (function () {
    function IconFeatherComponent() {
    }
    IconFeatherComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-feather',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-feather\">\n    <path d=\"M20.24 12.24a6 6 0 0 0-8.49-8.49L5 10.5V19h8.5z\"></path><line x1=\"16\" y1=\"8\" x2=\"2\" y2=\"22\"></line><line x1=\"17.5\" y1=\"15\" x2=\"9\" y2=\"15\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconFeatherComponent;
}());
export { IconFeatherComponent };
var IconFeather = /** @class */ (function () {
    function IconFeather() {
    }
    IconFeather.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconFeatherComponent],
                    exports: [IconFeatherComponent]
                },] }
    ];
    return IconFeather;
}());
export { IconFeather };
