// angular-feather/esm5/feather/file-plus.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconFilePlusComponent = /** @class */ (function () {
    function IconFilePlusComponent() {
    }
    IconFilePlusComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-file-plus',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-file-plus\">\n    <path d=\"M14 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V8z\"></path><polyline points=\"14 2 14 8 20 8\"></polyline><line x1=\"12\" y1=\"18\" x2=\"12\" y2=\"12\"></line><line x1=\"9\" y1=\"15\" x2=\"15\" y2=\"15\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconFilePlusComponent;
}());
export { IconFilePlusComponent };
var IconFilePlus = /** @class */ (function () {
    function IconFilePlus() {
    }
    IconFilePlus.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconFilePlusComponent],
                    exports: [IconFilePlusComponent]
                },] }
    ];
    return IconFilePlus;
}());
export { IconFilePlus };
