// angular-feather/esm5/feather/file.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconFileComponent = /** @class */ (function () {
    function IconFileComponent() {
    }
    IconFileComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-file',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-file\">\n    <path d=\"M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z\"></path><polyline points=\"13 2 13 9 20 9\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconFileComponent;
}());
export { IconFileComponent };
var IconFile = /** @class */ (function () {
    function IconFile() {
    }
    IconFile.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconFileComponent],
                    exports: [IconFileComponent]
                },] }
    ];
    return IconFile;
}());
export { IconFile };
