// angular-feather/esm5/feather/flag.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconFlagComponent = /** @class */ (function () {
    function IconFlagComponent() {
    }
    IconFlagComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-flag',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-flag\">\n    <path d=\"M4 15s1-1 4-1 5 2 8 2 4-1 4-1V3s-1 1-4 1-5-2-8-2-4 1-4 1z\"></path><line x1=\"4\" y1=\"22\" x2=\"4\" y2=\"15\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconFlagComponent;
}());
export { IconFlagComponent };
var IconFlag = /** @class */ (function () {
    function IconFlag() {
    }
    IconFlag.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconFlagComponent],
                    exports: [IconFlagComponent]
                },] }
    ];
    return IconFlag;
}());
export { IconFlag };
