// angular-feather/esm5/feather/folder-plus.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconFolderPlusComponent = /** @class */ (function () {
    function IconFolderPlusComponent() {
    }
    IconFolderPlusComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-folder-plus',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-folder-plus\">\n    <path d=\"M22 19a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h5l2 3h9a2 2 0 0 1 2 2z\"></path><line x1=\"12\" y1=\"11\" x2=\"12\" y2=\"17\"></line><line x1=\"9\" y1=\"14\" x2=\"15\" y2=\"14\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconFolderPlusComponent;
}());
export { IconFolderPlusComponent };
var IconFolderPlus = /** @class */ (function () {
    function IconFolderPlus() {
    }
    IconFolderPlus.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconFolderPlusComponent],
                    exports: [IconFolderPlusComponent]
                },] }
    ];
    return IconFolderPlus;
}());
export { IconFolderPlus };
