// angular-feather/esm5/feather/folder-minus.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconFolderMinusComponent = /** @class */ (function () {
    function IconFolderMinusComponent() {
    }
    IconFolderMinusComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-folder-minus',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-folder-minus\">\n    <path d=\"M22 19a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h5l2 3h9a2 2 0 0 1 2 2z\"></path><line x1=\"9\" y1=\"14\" x2=\"15\" y2=\"14\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconFolderMinusComponent;
}());
export { IconFolderMinusComponent };
var IconFolderMinus = /** @class */ (function () {
    function IconFolderMinus() {
    }
    IconFolderMinus.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconFolderMinusComponent],
                    exports: [IconFolderMinusComponent]
                },] }
    ];
    return IconFolderMinus;
}());
export { IconFolderMinus };
