// angular-feather/esm5/feather/folder.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconFolderComponent = /** @class */ (function () {
    function IconFolderComponent() {
    }
    IconFolderComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-folder',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-folder\">\n    <path d=\"M22 19a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h5l2 3h9a2 2 0 0 1 2 2z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconFolderComponent;
}());
export { IconFolderComponent };
var IconFolder = /** @class */ (function () {
    function IconFolder() {
    }
    IconFolder.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconFolderComponent],
                    exports: [IconFolderComponent]
                },] }
    ];
    return IconFolder;
}());
export { IconFolder };
