// angular-feather/esm5/feather/frown.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconFrownComponent = /** @class */ (function () {
    function IconFrownComponent() {
    }
    IconFrownComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-frown',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-frown\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><path d=\"M16 16s-1.5-2-4-2-4 2-4 2\"></path><line x1=\"9\" y1=\"9\" x2=\"9.01\" y2=\"9\"></line><line x1=\"15\" y1=\"9\" x2=\"15.01\" y2=\"9\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconFrownComponent;
}());
export { IconFrownComponent };
var IconFrown = /** @class */ (function () {
    function IconFrown() {
    }
    IconFrown.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconFrownComponent],
                    exports: [IconFrownComponent]
                },] }
    ];
    return IconFrown;
}());
export { IconFrown };
