// angular-feather/esm5/feather/gift.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconGiftComponent = /** @class */ (function () {
    function IconGiftComponent() {
    }
    IconGiftComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-gift',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-gift\">\n    <polyline points=\"20 12 20 22 4 22 4 12\"></polyline><rect x=\"2\" y=\"7\" width=\"20\" height=\"5\"></rect><line x1=\"12\" y1=\"22\" x2=\"12\" y2=\"7\"></line><path d=\"M12 7H7.5a2.5 2.5 0 0 1 0-5C11 2 12 7 12 7z\"></path><path d=\"M12 7h4.5a2.5 2.5 0 0 0 0-5C13 2 12 7 12 7z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconGiftComponent;
}());
export { IconGiftComponent };
var IconGift = /** @class */ (function () {
    function IconGift() {
    }
    IconGift.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconGiftComponent],
                    exports: [IconGiftComponent]
                },] }
    ];
    return IconGift;
}());
export { IconGift };
