// angular-feather/esm5/feather/git-branch.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconGitBranchComponent = /** @class */ (function () {
    function IconGitBranchComponent() {
    }
    IconGitBranchComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-git-branch',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-git-branch\">\n    <line x1=\"6\" y1=\"3\" x2=\"6\" y2=\"15\"></line><circle cx=\"18\" cy=\"6\" r=\"3\"></circle><circle cx=\"6\" cy=\"18\" r=\"3\"></circle><path d=\"M18 9a9 9 0 0 1-9 9\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconGitBranchComponent;
}());
export { IconGitBranchComponent };
var IconGitBranch = /** @class */ (function () {
    function IconGitBranch() {
    }
    IconGitBranch.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconGitBranchComponent],
                    exports: [IconGitBranchComponent]
                },] }
    ];
    return IconGitBranch;
}());
export { IconGitBranch };
