// angular-feather/esm5/feather/git-merge.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconGitMergeComponent = /** @class */ (function () {
    function IconGitMergeComponent() {
    }
    IconGitMergeComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-git-merge',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-git-merge\">\n    <circle cx=\"18\" cy=\"18\" r=\"3\"></circle><circle cx=\"6\" cy=\"6\" r=\"3\"></circle><path d=\"M6 21V9a9 9 0 0 0 9 9\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconGitMergeComponent;
}());
export { IconGitMergeComponent };
var IconGitMerge = /** @class */ (function () {
    function IconGitMerge() {
    }
    IconGitMerge.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconGitMergeComponent],
                    exports: [IconGitMergeComponent]
                },] }
    ];
    return IconGitMerge;
}());
export { IconGitMerge };
