// angular-feather/esm5/feather/git-pull-request.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconGitPullRequestComponent = /** @class */ (function () {
    function IconGitPullRequestComponent() {
    }
    IconGitPullRequestComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-git-pull-request',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-git-pull-request\">\n    <circle cx=\"18\" cy=\"18\" r=\"3\"></circle><circle cx=\"6\" cy=\"6\" r=\"3\"></circle><path d=\"M13 6h3a2 2 0 0 1 2 2v7\"></path><line x1=\"6\" y1=\"9\" x2=\"6\" y2=\"21\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconGitPullRequestComponent;
}());
export { IconGitPullRequestComponent };
var IconGitPullRequest = /** @class */ (function () {
    function IconGitPullRequest() {
    }
    IconGitPullRequest.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconGitPullRequestComponent],
                    exports: [IconGitPullRequestComponent]
                },] }
    ];
    return IconGitPullRequest;
}());
export { IconGitPullRequest };
