// angular-feather/esm5/feather/git-commit.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconGitCommitComponent = /** @class */ (function () {
    function IconGitCommitComponent() {
    }
    IconGitCommitComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-git-commit',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-git-commit\">\n    <circle cx=\"12\" cy=\"12\" r=\"4\"></circle><line x1=\"1.05\" y1=\"12\" x2=\"7\" y2=\"12\"></line><line x1=\"17.01\" y1=\"12\" x2=\"22.96\" y2=\"12\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconGitCommitComponent;
}());
export { IconGitCommitComponent };
var IconGitCommit = /** @class */ (function () {
    function IconGitCommit() {
    }
    IconGitCommit.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconGitCommitComponent],
                    exports: [IconGitCommitComponent]
                },] }
    ];
    return IconGitCommit;
}());
export { IconGitCommit };
