// angular-feather/esm5/feather/gitlab.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconGitlabComponent = /** @class */ (function () {
    function IconGitlabComponent() {
    }
    IconGitlabComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-gitlab',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-gitlab\">\n    <path d=\"M22.65 14.39L12 22.13 1.35 14.39a.84.84 0 0 1-.3-.94l1.22-3.78 2.44-7.51A.42.42 0 0 1 4.82 2a.43.43 0 0 1 .58 0 .42.42 0 0 1 .11.18l2.44 7.49h8.1l2.44-7.51A.42.42 0 0 1 18.6 2a.43.43 0 0 1 .58 0 .42.42 0 0 1 .11.18l2.44 7.51L23 13.45a.84.84 0 0 1-.35.94z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconGitlabComponent;
}());
export { IconGitlabComponent };
var IconGitlab = /** @class */ (function () {
    function IconGitlab() {
    }
    IconGitlab.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconGitlabComponent],
                    exports: [IconGitlabComponent]
                },] }
    ];
    return IconGitlab;
}());
export { IconGitlab };
