// angular-feather/esm5/feather/globe.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconGlobeComponent = /** @class */ (function () {
    function IconGlobeComponent() {
    }
    IconGlobeComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-globe',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-globe\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><line x1=\"2\" y1=\"12\" x2=\"22\" y2=\"12\"></line><path d=\"M12 2a15.3 15.3 0 0 1 4 10 15.3 15.3 0 0 1-4 10 15.3 15.3 0 0 1-4-10 15.3 15.3 0 0 1 4-10z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconGlobeComponent;
}());
export { IconGlobeComponent };
var IconGlobe = /** @class */ (function () {
    function IconGlobe() {
    }
    IconGlobe.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconGlobeComponent],
                    exports: [IconGlobeComponent]
                },] }
    ];
    return IconGlobe;
}());
export { IconGlobe };
