// angular-feather/esm5/feather/hard-drive.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconHardDriveComponent = /** @class */ (function () {
    function IconHardDriveComponent() {
    }
    IconHardDriveComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-hard-drive',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-hard-drive\">\n    <line x1=\"22\" y1=\"12\" x2=\"2\" y2=\"12\"></line><path d=\"M5.45 5.11L2 12v6a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2v-6l-3.45-6.89A2 2 0 0 0 16.76 4H7.24a2 2 0 0 0-1.79 1.11z\"></path><line x1=\"6\" y1=\"16\" x2=\"6\" y2=\"16\"></line><line x1=\"10\" y1=\"16\" x2=\"10\" y2=\"16\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconHardDriveComponent;
}());
export { IconHardDriveComponent };
var IconHardDrive = /** @class */ (function () {
    function IconHardDrive() {
    }
    IconHardDrive.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconHardDriveComponent],
                    exports: [IconHardDriveComponent]
                },] }
    ];
    return IconHardDrive;
}());
export { IconHardDrive };
