// angular-feather/esm5/feather/hash.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconHashComponent = /** @class */ (function () {
    function IconHashComponent() {
    }
    IconHashComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-hash',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-hash\">\n    <line x1=\"4\" y1=\"9\" x2=\"20\" y2=\"9\"></line><line x1=\"4\" y1=\"15\" x2=\"20\" y2=\"15\"></line><line x1=\"10\" y1=\"3\" x2=\"8\" y2=\"21\"></line><line x1=\"16\" y1=\"3\" x2=\"14\" y2=\"21\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconHashComponent;
}());
export { IconHashComponent };
var IconHash = /** @class */ (function () {
    function IconHash() {
    }
    IconHash.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconHashComponent],
                    exports: [IconHashComponent]
                },] }
    ];
    return IconHash;
}());
export { IconHash };
