// angular-feather/esm5/feather/headphones.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconHeadphonesComponent = /** @class */ (function () {
    function IconHeadphonesComponent() {
    }
    IconHeadphonesComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-headphones',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-headphones\">\n    <path d=\"M3 18v-6a9 9 0 0 1 18 0v6\"></path><path d=\"M21 19a2 2 0 0 1-2 2h-1a2 2 0 0 1-2-2v-3a2 2 0 0 1 2-2h3zM3 19a2 2 0 0 0 2 2h1a2 2 0 0 0 2-2v-3a2 2 0 0 0-2-2H3z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconHeadphonesComponent;
}());
export { IconHeadphonesComponent };
var IconHeadphones = /** @class */ (function () {
    function IconHeadphones() {
    }
    IconHeadphones.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconHeadphonesComponent],
                    exports: [IconHeadphonesComponent]
                },] }
    ];
    return IconHeadphones;
}());
export { IconHeadphones };
