// angular-feather/esm5/feather/help-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconHelpCircleComponent = /** @class */ (function () {
    function IconHelpCircleComponent() {
    }
    IconHelpCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-help-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-help-circle\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><path d=\"M9.09 9a3 3 0 0 1 5.83 1c0 2-3 3-3 3\"></path><line x1=\"12\" y1=\"17\" x2=\"12\" y2=\"17\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconHelpCircleComponent;
}());
export { IconHelpCircleComponent };
var IconHelpCircle = /** @class */ (function () {
    function IconHelpCircle() {
    }
    IconHelpCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconHelpCircleComponent],
                    exports: [IconHelpCircleComponent]
                },] }
    ];
    return IconHelpCircle;
}());
export { IconHelpCircle };
