// angular-feather/esm5/feather/home.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconHomeComponent = /** @class */ (function () {
    function IconHomeComponent() {
    }
    IconHomeComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-home',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-home\">\n    <path d=\"M3 9l9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z\"></path><polyline points=\"9 22 9 12 15 12 15 22\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconHomeComponent;
}());
export { IconHomeComponent };
var IconHome = /** @class */ (function () {
    function IconHome() {
    }
    IconHome.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconHomeComponent],
                    exports: [IconHomeComponent]
                },] }
    ];
    return IconHome;
}());
export { IconHome };
