// angular-feather/esm5/feather/image.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconImageComponent = /** @class */ (function () {
    function IconImageComponent() {
    }
    IconImageComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-image',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-image\">\n    <rect x=\"3\" y=\"3\" width=\"18\" height=\"18\" rx=\"2\" ry=\"2\"></rect><circle cx=\"8.5\" cy=\"8.5\" r=\"1.5\"></circle><polyline points=\"21 15 16 10 5 21\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconImageComponent;
}());
export { IconImageComponent };
var IconImage = /** @class */ (function () {
    function IconImage() {
    }
    IconImage.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconImageComponent],
                    exports: [IconImageComponent]
                },] }
    ];
    return IconImage;
}());
export { IconImage };
