// angular-feather/esm5/feather/inbox.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconInboxComponent = /** @class */ (function () {
    function IconInboxComponent() {
    }
    IconInboxComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-inbox',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-inbox\">\n    <polyline points=\"22 12 16 12 14 15 10 15 8 12 2 12\"></polyline><path d=\"M5.45 5.11L2 12v6a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2v-6l-3.45-6.89A2 2 0 0 0 16.76 4H7.24a2 2 0 0 0-1.79 1.11z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconInboxComponent;
}());
export { IconInboxComponent };
var IconInbox = /** @class */ (function () {
    function IconInbox() {
    }
    IconInbox.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconInboxComponent],
                    exports: [IconInboxComponent]
                },] }
    ];
    return IconInbox;
}());
export { IconInbox };
