// angular-feather/esm5/feather/info.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconInfoComponent = /** @class */ (function () {
    function IconInfoComponent() {
    }
    IconInfoComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-info',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-info\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><line x1=\"12\" y1=\"16\" x2=\"12\" y2=\"12\"></line><line x1=\"12\" y1=\"8\" x2=\"12\" y2=\"8\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconInfoComponent;
}());
export { IconInfoComponent };
var IconInfo = /** @class */ (function () {
    function IconInfo() {
    }
    IconInfo.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconInfoComponent],
                    exports: [IconInfoComponent]
                },] }
    ];
    return IconInfo;
}());
export { IconInfo };
