// angular-feather/esm5/feather/instagram.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconInstagramComponent = /** @class */ (function () {
    function IconInstagramComponent() {
    }
    IconInstagramComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-instagram',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-instagram\">\n    <rect x=\"2\" y=\"2\" width=\"20\" height=\"20\" rx=\"5\" ry=\"5\"></rect><path d=\"M16 11.37A4 4 0 1 1 12.63 8 4 4 0 0 1 16 11.37z\"></path><line x1=\"17.5\" y1=\"6.5\" x2=\"17.5\" y2=\"6.5\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconInstagramComponent;
}());
export { IconInstagramComponent };
var IconInstagram = /** @class */ (function () {
    function IconInstagram() {
    }
    IconInstagram.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconInstagramComponent],
                    exports: [IconInstagramComponent]
                },] }
    ];
    return IconInstagram;
}());
export { IconInstagram };
