// angular-feather/esm5/feather/italic.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconItalicComponent = /** @class */ (function () {
    function IconItalicComponent() {
    }
    IconItalicComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-italic',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-italic\">\n    <line x1=\"19\" y1=\"4\" x2=\"10\" y2=\"4\"></line><line x1=\"14\" y1=\"20\" x2=\"5\" y2=\"20\"></line><line x1=\"15\" y1=\"4\" x2=\"9\" y2=\"20\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconItalicComponent;
}());
export { IconItalicComponent };
var IconItalic = /** @class */ (function () {
    function IconItalic() {
    }
    IconItalic.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconItalicComponent],
                    exports: [IconItalicComponent]
                },] }
    ];
    return IconItalic;
}());
export { IconItalic };
