// angular-feather/esm5/feather/key.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconKeyComponent = /** @class */ (function () {
    function IconKeyComponent() {
    }
    IconKeyComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-key',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-key\">\n    <path d=\"M21 2l-2 2m-7.61 7.61a5.5 5.5 0 1 1-7.778 7.778 5.5 5.5 0 0 1 7.777-7.777zm0 0L15.5 7.5m0 0l3 3L22 7l-3-3m-3.5 3.5L19 4\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconKeyComponent;
}());
export { IconKeyComponent };
var IconKey = /** @class */ (function () {
    function IconKey() {
    }
    IconKey.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconKeyComponent],
                    exports: [IconKeyComponent]
                },] }
    ];
    return IconKey;
}());
export { IconKey };
