// angular-feather/esm5/feather/layers.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconLayersComponent = /** @class */ (function () {
    function IconLayersComponent() {
    }
    IconLayersComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-layers',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-layers\">\n    <polygon points=\"12 2 2 7 12 12 22 7 12 2\"></polygon><polyline points=\"2 17 12 22 22 17\"></polyline><polyline points=\"2 12 12 17 22 12\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconLayersComponent;
}());
export { IconLayersComponent };
var IconLayers = /** @class */ (function () {
    function IconLayers() {
    }
    IconLayers.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconLayersComponent],
                    exports: [IconLayersComponent]
                },] }
    ];
    return IconLayers;
}());
export { IconLayers };
