// angular-feather/esm5/feather/life-buoy.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconLifeBuoyComponent = /** @class */ (function () {
    function IconLifeBuoyComponent() {
    }
    IconLifeBuoyComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-life-buoy',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-life-buoy\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><circle cx=\"12\" cy=\"12\" r=\"4\"></circle><line x1=\"4.93\" y1=\"4.93\" x2=\"9.17\" y2=\"9.17\"></line><line x1=\"14.83\" y1=\"14.83\" x2=\"19.07\" y2=\"19.07\"></line><line x1=\"14.83\" y1=\"9.17\" x2=\"19.07\" y2=\"4.93\"></line><line x1=\"14.83\" y1=\"9.17\" x2=\"18.36\" y2=\"5.64\"></line><line x1=\"4.93\" y1=\"19.07\" x2=\"9.17\" y2=\"14.83\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconLifeBuoyComponent;
}());
export { IconLifeBuoyComponent };
var IconLifeBuoy = /** @class */ (function () {
    function IconLifeBuoy() {
    }
    IconLifeBuoy.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconLifeBuoyComponent],
                    exports: [IconLifeBuoyComponent]
                },] }
    ];
    return IconLifeBuoy;
}());
export { IconLifeBuoy };
