// angular-feather/esm5/feather/linkedin.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconLinkedinComponent = /** @class */ (function () {
    function IconLinkedinComponent() {
    }
    IconLinkedinComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-linkedin',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-linkedin\">\n    <path d=\"M16 8a6 6 0 0 1 6 6v7h-4v-7a2 2 0 0 0-2-2 2 2 0 0 0-2 2v7h-4v-7a6 6 0 0 1 6-6z\"></path><rect x=\"2\" y=\"9\" width=\"4\" height=\"12\"></rect><circle cx=\"4\" cy=\"4\" r=\"2\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconLinkedinComponent;
}());
export { IconLinkedinComponent };
var IconLinkedin = /** @class */ (function () {
    function IconLinkedin() {
    }
    IconLinkedin.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconLinkedinComponent],
                    exports: [IconLinkedinComponent]
                },] }
    ];
    return IconLinkedin;
}());
export { IconLinkedin };
