// angular-feather/esm5/feather/link.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconLinkComponent = /** @class */ (function () {
    function IconLinkComponent() {
    }
    IconLinkComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-link',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-link\">\n    <path d=\"M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71\"></path><path d=\"M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconLinkComponent;
}());
export { IconLinkComponent };
var IconLink = /** @class */ (function () {
    function IconLink() {
    }
    IconLink.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconLinkComponent],
                    exports: [IconLinkComponent]
                },] }
    ];
    return IconLink;
}());
export { IconLink };
