// angular-feather/esm5/feather/link-2.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconLink2Component = /** @class */ (function () {
    function IconLink2Component() {
    }
    IconLink2Component.decorators = [
        { type: Component, args: [{
                    selector: 'i-link-2',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-link-2\">\n    <path d=\"M15 7h3a5 5 0 0 1 5 5 5 5 0 0 1-5 5h-3m-6 0H6a5 5 0 0 1-5-5 5 5 0 0 1 5-5h3\"></path><line x1=\"8\" y1=\"12\" x2=\"16\" y2=\"12\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconLink2Component;
}());
export { IconLink2Component };
var IconLink2 = /** @class */ (function () {
    function IconLink2() {
    }
    IconLink2.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconLink2Component],
                    exports: [IconLink2Component]
                },] }
    ];
    return IconLink2;
}());
export { IconLink2 };
