// angular-feather/esm5/feather/lock.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconLockComponent = /** @class */ (function () {
    function IconLockComponent() {
    }
    IconLockComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-lock',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-lock\">\n    <rect x=\"3\" y=\"11\" width=\"18\" height=\"11\" rx=\"2\" ry=\"2\"></rect><path d=\"M7 11V7a5 5 0 0 1 10 0v4\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconLockComponent;
}());
export { IconLockComponent };
var IconLock = /** @class */ (function () {
    function IconLock() {
    }
    IconLock.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconLockComponent],
                    exports: [IconLockComponent]
                },] }
    ];
    return IconLock;
}());
export { IconLock };
