// angular-feather/esm5/feather/log-in.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconLogInComponent = /** @class */ (function () {
    function IconLogInComponent() {
    }
    IconLogInComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-log-in',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-log-in\">\n    <path d=\"M15 3h4a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2h-4\"></path><polyline points=\"10 17 15 12 10 7\"></polyline><line x1=\"15\" y1=\"12\" x2=\"3\" y2=\"12\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconLogInComponent;
}());
export { IconLogInComponent };
var IconLogIn = /** @class */ (function () {
    function IconLogIn() {
    }
    IconLogIn.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconLogInComponent],
                    exports: [IconLogInComponent]
                },] }
    ];
    return IconLogIn;
}());
export { IconLogIn };
