// angular-feather/esm5/feather/mail.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMailComponent = /** @class */ (function () {
    function IconMailComponent() {
    }
    IconMailComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-mail',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-mail\">\n    <path d=\"M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z\"></path><polyline points=\"22,6 12,13 2,6\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMailComponent;
}());
export { IconMailComponent };
var IconMail = /** @class */ (function () {
    function IconMail() {
    }
    IconMail.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMailComponent],
                    exports: [IconMailComponent]
                },] }
    ];
    return IconMail;
}());
export { IconMail };
