// angular-feather/esm5/feather/map-pin.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMapPinComponent = /** @class */ (function () {
    function IconMapPinComponent() {
    }
    IconMapPinComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-map-pin',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-map-pin\">\n    <path d=\"M21 10c0 7-9 13-9 13s-9-6-9-13a9 9 0 0 1 18 0z\"></path><circle cx=\"12\" cy=\"10\" r=\"3\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMapPinComponent;
}());
export { IconMapPinComponent };
var IconMapPin = /** @class */ (function () {
    function IconMapPin() {
    }
    IconMapPin.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMapPinComponent],
                    exports: [IconMapPinComponent]
                },] }
    ];
    return IconMapPin;
}());
export { IconMapPin };
