// angular-feather/esm5/feather/map.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMapComponent = /** @class */ (function () {
    function IconMapComponent() {
    }
    IconMapComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-map',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-map\">\n    <polygon points=\"1 6 1 22 8 18 16 22 23 18 23 2 16 6 8 2 1 6\"></polygon><line x1=\"8\" y1=\"2\" x2=\"8\" y2=\"18\"></line><line x1=\"16\" y1=\"6\" x2=\"16\" y2=\"22\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMapComponent;
}());
export { IconMapComponent };
var IconMap = /** @class */ (function () {
    function IconMap() {
    }
    IconMap.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMapComponent],
                    exports: [IconMapComponent]
                },] }
    ];
    return IconMap;
}());
export { IconMap };
