// angular-feather/esm5/feather/maximize-2.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMaximize2Component = /** @class */ (function () {
    function IconMaximize2Component() {
    }
    IconMaximize2Component.decorators = [
        { type: Component, args: [{
                    selector: 'i-maximize-2',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-maximize-2\">\n    <polyline points=\"15 3 21 3 21 9\"></polyline><polyline points=\"9 21 3 21 3 15\"></polyline><line x1=\"21\" y1=\"3\" x2=\"14\" y2=\"10\"></line><line x1=\"3\" y1=\"21\" x2=\"10\" y2=\"14\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMaximize2Component;
}());
export { IconMaximize2Component };
var IconMaximize2 = /** @class */ (function () {
    function IconMaximize2() {
    }
    IconMaximize2.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMaximize2Component],
                    exports: [IconMaximize2Component]
                },] }
    ];
    return IconMaximize2;
}());
export { IconMaximize2 };
