// angular-feather/esm5/feather/maximize.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMaximizeComponent = /** @class */ (function () {
    function IconMaximizeComponent() {
    }
    IconMaximizeComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-maximize',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-maximize\">\n    <path d=\"M8 3H5a2 2 0 0 0-2 2v3m18 0V5a2 2 0 0 0-2-2h-3m0 18h3a2 2 0 0 0 2-2v-3M3 16v3a2 2 0 0 0 2 2h3\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMaximizeComponent;
}());
export { IconMaximizeComponent };
var IconMaximize = /** @class */ (function () {
    function IconMaximize() {
    }
    IconMaximize.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMaximizeComponent],
                    exports: [IconMaximizeComponent]
                },] }
    ];
    return IconMaximize;
}());
export { IconMaximize };
