// angular-feather/esm5/feather/meh.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMehComponent = /** @class */ (function () {
    function IconMehComponent() {
    }
    IconMehComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-meh',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-meh\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><line x1=\"8\" y1=\"15\" x2=\"16\" y2=\"15\"></line><line x1=\"9\" y1=\"9\" x2=\"9.01\" y2=\"9\"></line><line x1=\"15\" y1=\"9\" x2=\"15.01\" y2=\"9\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMehComponent;
}());
export { IconMehComponent };
var IconMeh = /** @class */ (function () {
    function IconMeh() {
    }
    IconMeh.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMehComponent],
                    exports: [IconMehComponent]
                },] }
    ];
    return IconMeh;
}());
export { IconMeh };
