// angular-feather/esm5/feather/menu.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMenuComponent = /** @class */ (function () {
    function IconMenuComponent() {
    }
    IconMenuComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-menu',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-menu\">\n    <line x1=\"3\" y1=\"12\" x2=\"21\" y2=\"12\"></line><line x1=\"3\" y1=\"6\" x2=\"21\" y2=\"6\"></line><line x1=\"3\" y1=\"18\" x2=\"21\" y2=\"18\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMenuComponent;
}());
export { IconMenuComponent };
var IconMenu = /** @class */ (function () {
    function IconMenu() {
    }
    IconMenu.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMenuComponent],
                    exports: [IconMenuComponent]
                },] }
    ];
    return IconMenu;
}());
export { IconMenu };
