// angular-feather/esm5/feather/message-square.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMessageSquareComponent = /** @class */ (function () {
    function IconMessageSquareComponent() {
    }
    IconMessageSquareComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-message-square',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-message-square\">\n    <path d=\"M21 15a2 2 0 0 1-2 2H7l-4 4V5a2 2 0 0 1 2-2h14a2 2 0 0 1 2 2z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMessageSquareComponent;
}());
export { IconMessageSquareComponent };
var IconMessageSquare = /** @class */ (function () {
    function IconMessageSquare() {
    }
    IconMessageSquare.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMessageSquareComponent],
                    exports: [IconMessageSquareComponent]
                },] }
    ];
    return IconMessageSquare;
}());
export { IconMessageSquare };
