// angular-feather/esm5/feather/message-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMessageCircleComponent = /** @class */ (function () {
    function IconMessageCircleComponent() {
    }
    IconMessageCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-message-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-message-circle\">\n    <path d=\"M21 11.5a8.38 8.38 0 0 1-.9 3.8 8.5 8.5 0 0 1-7.6 4.7 8.38 8.38 0 0 1-3.8-.9L3 21l1.9-5.7a8.38 8.38 0 0 1-.9-3.8 8.5 8.5 0 0 1 4.7-7.6 8.38 8.38 0 0 1 3.8-.9h.5a8.48 8.48 0 0 1 8 8v.5z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMessageCircleComponent;
}());
export { IconMessageCircleComponent };
var IconMessageCircle = /** @class */ (function () {
    function IconMessageCircle() {
    }
    IconMessageCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMessageCircleComponent],
                    exports: [IconMessageCircleComponent]
                },] }
    ];
    return IconMessageCircle;
}());
export { IconMessageCircle };
