// angular-feather/esm5/feather/mic-off.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMicOffComponent = /** @class */ (function () {
    function IconMicOffComponent() {
    }
    IconMicOffComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-mic-off',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-mic-off\">\n    <line x1=\"1\" y1=\"1\" x2=\"23\" y2=\"23\"></line><path d=\"M9 9v3a3 3 0 0 0 5.12 2.12M15 9.34V4a3 3 0 0 0-5.94-.6\"></path><path d=\"M17 16.95A7 7 0 0 1 5 12v-2m14 0v2a7 7 0 0 1-.11 1.23\"></path><line x1=\"12\" y1=\"19\" x2=\"12\" y2=\"23\"></line><line x1=\"8\" y1=\"23\" x2=\"16\" y2=\"23\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMicOffComponent;
}());
export { IconMicOffComponent };
var IconMicOff = /** @class */ (function () {
    function IconMicOff() {
    }
    IconMicOff.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMicOffComponent],
                    exports: [IconMicOffComponent]
                },] }
    ];
    return IconMicOff;
}());
export { IconMicOff };
