// angular-feather/esm5/feather/mic.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMicComponent = /** @class */ (function () {
    function IconMicComponent() {
    }
    IconMicComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-mic',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-mic\">\n    <path d=\"M12 1a3 3 0 0 0-3 3v8a3 3 0 0 0 6 0V4a3 3 0 0 0-3-3z\"></path><path d=\"M19 10v2a7 7 0 0 1-14 0v-2\"></path><line x1=\"12\" y1=\"19\" x2=\"12\" y2=\"23\"></line><line x1=\"8\" y1=\"23\" x2=\"16\" y2=\"23\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMicComponent;
}());
export { IconMicComponent };
var IconMic = /** @class */ (function () {
    function IconMic() {
    }
    IconMic.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMicComponent],
                    exports: [IconMicComponent]
                },] }
    ];
    return IconMic;
}());
export { IconMic };
