// angular-feather/esm5/feather/minimize-2.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMinimize2Component = /** @class */ (function () {
    function IconMinimize2Component() {
    }
    IconMinimize2Component.decorators = [
        { type: Component, args: [{
                    selector: 'i-minimize-2',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-minimize-2\">\n    <polyline points=\"4 14 10 14 10 20\"></polyline><polyline points=\"20 10 14 10 14 4\"></polyline><line x1=\"14\" y1=\"10\" x2=\"21\" y2=\"3\"></line><line x1=\"3\" y1=\"21\" x2=\"10\" y2=\"14\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMinimize2Component;
}());
export { IconMinimize2Component };
var IconMinimize2 = /** @class */ (function () {
    function IconMinimize2() {
    }
    IconMinimize2.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMinimize2Component],
                    exports: [IconMinimize2Component]
                },] }
    ];
    return IconMinimize2;
}());
export { IconMinimize2 };
