// angular-feather/esm5/feather/minimize.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMinimizeComponent = /** @class */ (function () {
    function IconMinimizeComponent() {
    }
    IconMinimizeComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-minimize',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-minimize\">\n    <path d=\"M8 3v3a2 2 0 0 1-2 2H3m18 0h-3a2 2 0 0 1-2-2V3m0 18v-3a2 2 0 0 1 2-2h3M3 16h3a2 2 0 0 1 2 2v3\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMinimizeComponent;
}());
export { IconMinimizeComponent };
var IconMinimize = /** @class */ (function () {
    function IconMinimize() {
    }
    IconMinimize.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMinimizeComponent],
                    exports: [IconMinimizeComponent]
                },] }
    ];
    return IconMinimize;
}());
export { IconMinimize };
