// angular-feather/esm5/feather/minus-square.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMinusSquareComponent = /** @class */ (function () {
    function IconMinusSquareComponent() {
    }
    IconMinusSquareComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-minus-square',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-minus-square\">\n    <rect x=\"3\" y=\"3\" width=\"18\" height=\"18\" rx=\"2\" ry=\"2\"></rect><line x1=\"8\" y1=\"12\" x2=\"16\" y2=\"12\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMinusSquareComponent;
}());
export { IconMinusSquareComponent };
var IconMinusSquare = /** @class */ (function () {
    function IconMinusSquare() {
    }
    IconMinusSquare.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMinusSquareComponent],
                    exports: [IconMinusSquareComponent]
                },] }
    ];
    return IconMinusSquare;
}());
export { IconMinusSquare };
