// angular-feather/esm5/feather/monitor.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMonitorComponent = /** @class */ (function () {
    function IconMonitorComponent() {
    }
    IconMonitorComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-monitor',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-monitor\">\n    <rect x=\"2\" y=\"3\" width=\"20\" height=\"14\" rx=\"2\" ry=\"2\"></rect><line x1=\"8\" y1=\"21\" x2=\"16\" y2=\"21\"></line><line x1=\"12\" y1=\"17\" x2=\"12\" y2=\"21\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMonitorComponent;
}());
export { IconMonitorComponent };
var IconMonitor = /** @class */ (function () {
    function IconMonitor() {
    }
    IconMonitor.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMonitorComponent],
                    exports: [IconMonitorComponent]
                },] }
    ];
    return IconMonitor;
}());
export { IconMonitor };
