// angular-feather/esm5/feather/more-vertical.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMoreVerticalComponent = /** @class */ (function () {
    function IconMoreVerticalComponent() {
    }
    IconMoreVerticalComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-more-vertical',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-more-vertical\">\n    <circle cx=\"12\" cy=\"12\" r=\"1\"></circle><circle cx=\"12\" cy=\"5\" r=\"1\"></circle><circle cx=\"12\" cy=\"19\" r=\"1\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMoreVerticalComponent;
}());
export { IconMoreVerticalComponent };
var IconMoreVertical = /** @class */ (function () {
    function IconMoreVertical() {
    }
    IconMoreVertical.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMoreVerticalComponent],
                    exports: [IconMoreVerticalComponent]
                },] }
    ];
    return IconMoreVertical;
}());
export { IconMoreVertical };
