// angular-feather/esm5/feather/mouse-pointer.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMousePointerComponent = /** @class */ (function () {
    function IconMousePointerComponent() {
    }
    IconMousePointerComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-mouse-pointer',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-mouse-pointer\">\n    <path d=\"M3 3l7.07 16.97 2.51-7.39 7.39-2.51L3 3z\"></path><path d=\"M13 13l6 6\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMousePointerComponent;
}());
export { IconMousePointerComponent };
var IconMousePointer = /** @class */ (function () {
    function IconMousePointer() {
    }
    IconMousePointer.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMousePointerComponent],
                    exports: [IconMousePointerComponent]
                },] }
    ];
    return IconMousePointer;
}());
export { IconMousePointer };
