// angular-feather/esm5/feather/move.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMoveComponent = /** @class */ (function () {
    function IconMoveComponent() {
    }
    IconMoveComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-move',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-move\">\n    <polyline points=\"5 9 2 12 5 15\"></polyline><polyline points=\"9 5 12 2 15 5\"></polyline><polyline points=\"15 19 12 22 9 19\"></polyline><polyline points=\"19 9 22 12 19 15\"></polyline><line x1=\"2\" y1=\"12\" x2=\"22\" y2=\"12\"></line><line x1=\"12\" y1=\"2\" x2=\"12\" y2=\"22\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMoveComponent;
}());
export { IconMoveComponent };
var IconMove = /** @class */ (function () {
    function IconMove() {
    }
    IconMove.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMoveComponent],
                    exports: [IconMoveComponent]
                },] }
    ];
    return IconMove;
}());
export { IconMove };
