// angular-feather/esm5/feather/music.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconMusicComponent = /** @class */ (function () {
    function IconMusicComponent() {
    }
    IconMusicComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-music',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-music\">\n    <path d=\"M9 18V5l12-2v13\"></path><circle cx=\"6\" cy=\"18\" r=\"3\"></circle><circle cx=\"18\" cy=\"16\" r=\"3\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconMusicComponent;
}());
export { IconMusicComponent };
var IconMusic = /** @class */ (function () {
    function IconMusic() {
    }
    IconMusic.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconMusicComponent],
                    exports: [IconMusicComponent]
                },] }
    ];
    return IconMusic;
}());
export { IconMusic };
