// angular-feather/esm5/feather/navigation-2.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconNavigation2Component = /** @class */ (function () {
    function IconNavigation2Component() {
    }
    IconNavigation2Component.decorators = [
        { type: Component, args: [{
                    selector: 'i-navigation-2',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-navigation-2\">\n    <polygon points=\"12 2 19 21 12 17 5 21 12 2\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconNavigation2Component;
}());
export { IconNavigation2Component };
var IconNavigation2 = /** @class */ (function () {
    function IconNavigation2() {
    }
    IconNavigation2.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconNavigation2Component],
                    exports: [IconNavigation2Component]
                },] }
    ];
    return IconNavigation2;
}());
export { IconNavigation2 };
