// angular-feather/esm5/feather/navigation.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconNavigationComponent = /** @class */ (function () {
    function IconNavigationComponent() {
    }
    IconNavigationComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-navigation',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-navigation\">\n    <polygon points=\"3 11 22 2 13 21 11 13 3 11\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconNavigationComponent;
}());
export { IconNavigationComponent };
var IconNavigation = /** @class */ (function () {
    function IconNavigation() {
    }
    IconNavigation.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconNavigationComponent],
                    exports: [IconNavigationComponent]
                },] }
    ];
    return IconNavigation;
}());
export { IconNavigation };
