// angular-feather/esm5/feather/octagon.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconOctagonComponent = /** @class */ (function () {
    function IconOctagonComponent() {
    }
    IconOctagonComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-octagon',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-octagon\">\n    <polygon points=\"7.86 2 16.14 2 22 7.86 22 16.14 16.14 22 7.86 22 2 16.14 2 7.86 7.86 2\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconOctagonComponent;
}());
export { IconOctagonComponent };
var IconOctagon = /** @class */ (function () {
    function IconOctagon() {
    }
    IconOctagon.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconOctagonComponent],
                    exports: [IconOctagonComponent]
                },] }
    ];
    return IconOctagon;
}());
export { IconOctagon };
