// angular-feather/esm5/feather/paperclip.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPaperclipComponent = /** @class */ (function () {
    function IconPaperclipComponent() {
    }
    IconPaperclipComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-paperclip',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-paperclip\">\n    <path d=\"M21.44 11.05l-9.19 9.19a6 6 0 0 1-8.49-8.49l9.19-9.19a4 4 0 0 1 5.66 5.66l-9.2 9.19a2 2 0 0 1-2.83-2.83l8.49-8.48\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPaperclipComponent;
}());
export { IconPaperclipComponent };
var IconPaperclip = /** @class */ (function () {
    function IconPaperclip() {
    }
    IconPaperclip.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPaperclipComponent],
                    exports: [IconPaperclipComponent]
                },] }
    ];
    return IconPaperclip;
}());
export { IconPaperclip };
