// angular-feather/esm5/feather/pause.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPauseComponent = /** @class */ (function () {
    function IconPauseComponent() {
    }
    IconPauseComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-pause',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-pause\">\n    <rect x=\"6\" y=\"4\" width=\"4\" height=\"16\"></rect><rect x=\"14\" y=\"4\" width=\"4\" height=\"16\"></rect>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPauseComponent;
}());
export { IconPauseComponent };
var IconPause = /** @class */ (function () {
    function IconPause() {
    }
    IconPause.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPauseComponent],
                    exports: [IconPauseComponent]
                },] }
    ];
    return IconPause;
}());
export { IconPause };
