// angular-feather/esm5/feather/pause-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPauseCircleComponent = /** @class */ (function () {
    function IconPauseCircleComponent() {
    }
    IconPauseCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-pause-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-pause-circle\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><line x1=\"10\" y1=\"15\" x2=\"10\" y2=\"9\"></line><line x1=\"14\" y1=\"15\" x2=\"14\" y2=\"9\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPauseCircleComponent;
}());
export { IconPauseCircleComponent };
var IconPauseCircle = /** @class */ (function () {
    function IconPauseCircle() {
    }
    IconPauseCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPauseCircleComponent],
                    exports: [IconPauseCircleComponent]
                },] }
    ];
    return IconPauseCircle;
}());
export { IconPauseCircle };
