// angular-feather/esm5/feather/percent.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPercentComponent = /** @class */ (function () {
    function IconPercentComponent() {
    }
    IconPercentComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-percent',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-percent\">\n    <line x1=\"19\" y1=\"5\" x2=\"5\" y2=\"19\"></line><circle cx=\"6.5\" cy=\"6.5\" r=\"2.5\"></circle><circle cx=\"17.5\" cy=\"17.5\" r=\"2.5\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPercentComponent;
}());
export { IconPercentComponent };
var IconPercent = /** @class */ (function () {
    function IconPercent() {
    }
    IconPercent.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPercentComponent],
                    exports: [IconPercentComponent]
                },] }
    ];
    return IconPercent;
}());
export { IconPercent };
