// angular-feather/esm5/feather/phone-call.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPhoneCallComponent = /** @class */ (function () {
    function IconPhoneCallComponent() {
    }
    IconPhoneCallComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-phone-call',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-phone-call\">\n    <path d=\"M15.05 5A5 5 0 0 1 19 8.95M15.05 1A9 9 0 0 1 23 8.94m-1 7.98v3a2 2 0 0 1-2.18 2 19.79 19.79 0 0 1-8.63-3.07 19.5 19.5 0 0 1-6-6 19.79 19.79 0 0 1-3.07-8.67A2 2 0 0 1 4.11 2h3a2 2 0 0 1 2 1.72 12.84 12.84 0 0 0 .7 2.81 2 2 0 0 1-.45 2.11L8.09 9.91a16 16 0 0 0 6 6l1.27-1.27a2 2 0 0 1 2.11-.45 12.84 12.84 0 0 0 2.81.7A2 2 0 0 1 22 16.92z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPhoneCallComponent;
}());
export { IconPhoneCallComponent };
var IconPhoneCall = /** @class */ (function () {
    function IconPhoneCall() {
    }
    IconPhoneCall.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPhoneCallComponent],
                    exports: [IconPhoneCallComponent]
                },] }
    ];
    return IconPhoneCall;
}());
export { IconPhoneCall };
