// angular-feather/esm5/feather/play.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPlayComponent = /** @class */ (function () {
    function IconPlayComponent() {
    }
    IconPlayComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-play',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-play\">\n    <polygon points=\"5 3 19 12 5 21 5 3\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPlayComponent;
}());
export { IconPlayComponent };
var IconPlay = /** @class */ (function () {
    function IconPlay() {
    }
    IconPlay.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPlayComponent],
                    exports: [IconPlayComponent]
                },] }
    ];
    return IconPlay;
}());
export { IconPlay };
