// angular-feather/esm5/feather/play-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPlayCircleComponent = /** @class */ (function () {
    function IconPlayCircleComponent() {
    }
    IconPlayCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-play-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-play-circle\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><polygon points=\"10 8 16 12 10 16 10 8\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPlayCircleComponent;
}());
export { IconPlayCircleComponent };
var IconPlayCircle = /** @class */ (function () {
    function IconPlayCircle() {
    }
    IconPlayCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPlayCircleComponent],
                    exports: [IconPlayCircleComponent]
                },] }
    ];
    return IconPlayCircle;
}());
export { IconPlayCircle };
