// angular-feather/esm5/feather/plus-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPlusCircleComponent = /** @class */ (function () {
    function IconPlusCircleComponent() {
    }
    IconPlusCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-plus-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-plus-circle\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><line x1=\"12\" y1=\"8\" x2=\"12\" y2=\"16\"></line><line x1=\"8\" y1=\"12\" x2=\"16\" y2=\"12\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPlusCircleComponent;
}());
export { IconPlusCircleComponent };
var IconPlusCircle = /** @class */ (function () {
    function IconPlusCircle() {
    }
    IconPlusCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPlusCircleComponent],
                    exports: [IconPlusCircleComponent]
                },] }
    ];
    return IconPlusCircle;
}());
export { IconPlusCircle };
