// angular-feather/esm5/feather/plus.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPlusComponent = /** @class */ (function () {
    function IconPlusComponent() {
    }
    IconPlusComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-plus',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-plus\">\n    <line x1=\"12\" y1=\"5\" x2=\"12\" y2=\"19\"></line><line x1=\"5\" y1=\"12\" x2=\"19\" y2=\"12\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPlusComponent;
}());
export { IconPlusComponent };
var IconPlus = /** @class */ (function () {
    function IconPlus() {
    }
    IconPlus.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPlusComponent],
                    exports: [IconPlusComponent]
                },] }
    ];
    return IconPlus;
}());
export { IconPlus };
