// angular-feather/esm5/feather/plus-square.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPlusSquareComponent = /** @class */ (function () {
    function IconPlusSquareComponent() {
    }
    IconPlusSquareComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-plus-square',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-plus-square\">\n    <rect x=\"3\" y=\"3\" width=\"18\" height=\"18\" rx=\"2\" ry=\"2\"></rect><line x1=\"12\" y1=\"8\" x2=\"12\" y2=\"16\"></line><line x1=\"8\" y1=\"12\" x2=\"16\" y2=\"12\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPlusSquareComponent;
}());
export { IconPlusSquareComponent };
var IconPlusSquare = /** @class */ (function () {
    function IconPlusSquare() {
    }
    IconPlusSquare.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPlusSquareComponent],
                    exports: [IconPlusSquareComponent]
                },] }
    ];
    return IconPlusSquare;
}());
export { IconPlusSquare };
