// angular-feather/esm5/feather/pocket.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPocketComponent = /** @class */ (function () {
    function IconPocketComponent() {
    }
    IconPocketComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-pocket',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-pocket\">\n    <path d=\"M4 3h16a2 2 0 0 1 2 2v6a10 10 0 0 1-10 10A10 10 0 0 1 2 11V5a2 2 0 0 1 2-2z\"></path><polyline points=\"8 10 12 14 16 10\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPocketComponent;
}());
export { IconPocketComponent };
var IconPocket = /** @class */ (function () {
    function IconPocket() {
    }
    IconPocket.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPocketComponent],
                    exports: [IconPocketComponent]
                },] }
    ];
    return IconPocket;
}());
export { IconPocket };
