// angular-feather/esm5/feather/power.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPowerComponent = /** @class */ (function () {
    function IconPowerComponent() {
    }
    IconPowerComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-power',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-power\">\n    <path d=\"M18.36 6.64a9 9 0 1 1-12.73 0\"></path><line x1=\"12\" y1=\"2\" x2=\"12\" y2=\"12\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPowerComponent;
}());
export { IconPowerComponent };
var IconPower = /** @class */ (function () {
    function IconPower() {
    }
    IconPower.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPowerComponent],
                    exports: [IconPowerComponent]
                },] }
    ];
    return IconPower;
}());
export { IconPower };
