// angular-feather/esm5/feather/printer.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconPrinterComponent = /** @class */ (function () {
    function IconPrinterComponent() {
    }
    IconPrinterComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-printer',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-printer\">\n    <polyline points=\"6 9 6 2 18 2 18 9\"></polyline><path d=\"M6 18H4a2 2 0 0 1-2-2v-5a2 2 0 0 1 2-2h16a2 2 0 0 1 2 2v5a2 2 0 0 1-2 2h-2\"></path><rect x=\"6\" y=\"14\" width=\"12\" height=\"8\"></rect>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconPrinterComponent;
}());
export { IconPrinterComponent };
var IconPrinter = /** @class */ (function () {
    function IconPrinter() {
    }
    IconPrinter.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconPrinterComponent],
                    exports: [IconPrinterComponent]
                },] }
    ];
    return IconPrinter;
}());
export { IconPrinter };
