// angular-feather/esm5/feather/radio.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconRadioComponent = /** @class */ (function () {
    function IconRadioComponent() {
    }
    IconRadioComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-radio',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-radio\">\n    <circle cx=\"12\" cy=\"12\" r=\"2\"></circle><path d=\"M16.24 7.76a6 6 0 0 1 0 8.49m-8.48-.01a6 6 0 0 1 0-8.49m11.31-2.82a10 10 0 0 1 0 14.14m-14.14 0a10 10 0 0 1 0-14.14\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconRadioComponent;
}());
export { IconRadioComponent };
var IconRadio = /** @class */ (function () {
    function IconRadio() {
    }
    IconRadio.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconRadioComponent],
                    exports: [IconRadioComponent]
                },] }
    ];
    return IconRadio;
}());
export { IconRadio };
