// angular-feather/esm5/feather/refresh-cw.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconRefreshCwComponent = /** @class */ (function () {
    function IconRefreshCwComponent() {
    }
    IconRefreshCwComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-refresh-cw',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-refresh-cw\">\n    <polyline points=\"23 4 23 10 17 10\"></polyline><polyline points=\"1 20 1 14 7 14\"></polyline><path d=\"M3.51 9a9 9 0 0 1 14.85-3.36L23 10M1 14l4.64 4.36A9 9 0 0 0 20.49 15\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconRefreshCwComponent;
}());
export { IconRefreshCwComponent };
var IconRefreshCw = /** @class */ (function () {
    function IconRefreshCw() {
    }
    IconRefreshCw.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconRefreshCwComponent],
                    exports: [IconRefreshCwComponent]
                },] }
    ];
    return IconRefreshCw;
}());
export { IconRefreshCw };
