// angular-feather/esm5/feather/refresh-ccw.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconRefreshCcwComponent = /** @class */ (function () {
    function IconRefreshCcwComponent() {
    }
    IconRefreshCcwComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-refresh-ccw',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-refresh-ccw\">\n    <polyline points=\"1 4 1 10 7 10\"></polyline><polyline points=\"23 20 23 14 17 14\"></polyline><path d=\"M20.49 9A9 9 0 0 0 5.64 5.64L1 10m22 4l-4.64 4.36A9 9 0 0 1 3.51 15\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconRefreshCcwComponent;
}());
export { IconRefreshCcwComponent };
var IconRefreshCcw = /** @class */ (function () {
    function IconRefreshCcw() {
    }
    IconRefreshCcw.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconRefreshCcwComponent],
                    exports: [IconRefreshCcwComponent]
                },] }
    ];
    return IconRefreshCcw;
}());
export { IconRefreshCcw };
