// angular-feather/esm5/feather/repeat.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconRepeatComponent = /** @class */ (function () {
    function IconRepeatComponent() {
    }
    IconRepeatComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-repeat',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-repeat\">\n    <polyline points=\"17 1 21 5 17 9\"></polyline><path d=\"M3 11V9a4 4 0 0 1 4-4h14\"></path><polyline points=\"7 23 3 19 7 15\"></polyline><path d=\"M21 13v2a4 4 0 0 1-4 4H3\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconRepeatComponent;
}());
export { IconRepeatComponent };
var IconRepeat = /** @class */ (function () {
    function IconRepeat() {
    }
    IconRepeat.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconRepeatComponent],
                    exports: [IconRepeatComponent]
                },] }
    ];
    return IconRepeat;
}());
export { IconRepeat };
