// angular-feather/esm5/feather/rewind.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconRewindComponent = /** @class */ (function () {
    function IconRewindComponent() {
    }
    IconRewindComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-rewind',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-rewind\">\n    <polygon points=\"11 19 2 12 11 5 11 19\"></polygon><polygon points=\"22 19 13 12 22 5 22 19\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconRewindComponent;
}());
export { IconRewindComponent };
var IconRewind = /** @class */ (function () {
    function IconRewind() {
    }
    IconRewind.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconRewindComponent],
                    exports: [IconRewindComponent]
                },] }
    ];
    return IconRewind;
}());
export { IconRewind };
