// angular-feather/esm5/feather/rotate-cw.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconRotateCwComponent = /** @class */ (function () {
    function IconRotateCwComponent() {
    }
    IconRotateCwComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-rotate-cw',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-rotate-cw\">\n    <polyline points=\"23 4 23 10 17 10\"></polyline><path d=\"M20.49 15a9 9 0 1 1-2.12-9.36L23 10\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconRotateCwComponent;
}());
export { IconRotateCwComponent };
var IconRotateCw = /** @class */ (function () {
    function IconRotateCw() {
    }
    IconRotateCw.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconRotateCwComponent],
                    exports: [IconRotateCwComponent]
                },] }
    ];
    return IconRotateCw;
}());
export { IconRotateCw };
