// angular-feather/esm5/feather/rss.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconRssComponent = /** @class */ (function () {
    function IconRssComponent() {
    }
    IconRssComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-rss',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-rss\">\n    <path d=\"M4 11a9 9 0 0 1 9 9\"></path><path d=\"M4 4a16 16 0 0 1 16 16\"></path><circle cx=\"5\" cy=\"19\" r=\"1\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconRssComponent;
}());
export { IconRssComponent };
var IconRss = /** @class */ (function () {
    function IconRss() {
    }
    IconRss.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconRssComponent],
                    exports: [IconRssComponent]
                },] }
    ];
    return IconRss;
}());
export { IconRss };
