// angular-feather/esm5/feather/scissors.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconScissorsComponent = /** @class */ (function () {
    function IconScissorsComponent() {
    }
    IconScissorsComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-scissors',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-scissors\">\n    <circle cx=\"6\" cy=\"6\" r=\"3\"></circle><circle cx=\"6\" cy=\"18\" r=\"3\"></circle><line x1=\"20\" y1=\"4\" x2=\"8.12\" y2=\"15.88\"></line><line x1=\"14.47\" y1=\"14.48\" x2=\"20\" y2=\"20\"></line><line x1=\"8.12\" y1=\"8.12\" x2=\"12\" y2=\"12\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconScissorsComponent;
}());
export { IconScissorsComponent };
var IconScissors = /** @class */ (function () {
    function IconScissors() {
    }
    IconScissors.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconScissorsComponent],
                    exports: [IconScissorsComponent]
                },] }
    ];
    return IconScissors;
}());
export { IconScissors };
