// angular-feather/esm5/feather/search.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconSearchComponent = /** @class */ (function () {
    function IconSearchComponent() {
    }
    IconSearchComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-search',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-search\">\n    <circle cx=\"11\" cy=\"11\" r=\"8\"></circle><line x1=\"21\" y1=\"21\" x2=\"16.65\" y2=\"16.65\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconSearchComponent;
}());
export { IconSearchComponent };
var IconSearch = /** @class */ (function () {
    function IconSearch() {
    }
    IconSearch.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconSearchComponent],
                    exports: [IconSearchComponent]
                },] }
    ];
    return IconSearch;
}());
export { IconSearch };
