// angular-feather/esm5/feather/send.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconSendComponent = /** @class */ (function () {
    function IconSendComponent() {
    }
    IconSendComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-send',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-send\">\n    <line x1=\"22\" y1=\"2\" x2=\"11\" y2=\"13\"></line><polygon points=\"22 2 15 22 11 13 2 9 22 2\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconSendComponent;
}());
export { IconSendComponent };
var IconSend = /** @class */ (function () {
    function IconSend() {
    }
    IconSend.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconSendComponent],
                    exports: [IconSendComponent]
                },] }
    ];
    return IconSend;
}());
export { IconSend };
