// angular-feather/esm5/feather/server.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconServerComponent = /** @class */ (function () {
    function IconServerComponent() {
    }
    IconServerComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-server',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-server\">\n    <rect x=\"2\" y=\"2\" width=\"20\" height=\"8\" rx=\"2\" ry=\"2\"></rect><rect x=\"2\" y=\"14\" width=\"20\" height=\"8\" rx=\"2\" ry=\"2\"></rect><line x1=\"6\" y1=\"6\" x2=\"6\" y2=\"6\"></line><line x1=\"6\" y1=\"18\" x2=\"6\" y2=\"18\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconServerComponent;
}());
export { IconServerComponent };
var IconServer = /** @class */ (function () {
    function IconServer() {
    }
    IconServer.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconServerComponent],
                    exports: [IconServerComponent]
                },] }
    ];
    return IconServer;
}());
export { IconServer };
