// angular-feather/esm5/feather/share.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconShareComponent = /** @class */ (function () {
    function IconShareComponent() {
    }
    IconShareComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-share',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-share\">\n    <path d=\"M4 12v8a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2v-8\"></path><polyline points=\"16 6 12 2 8 6\"></polyline><line x1=\"12\" y1=\"2\" x2=\"12\" y2=\"15\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconShareComponent;
}());
export { IconShareComponent };
var IconShare = /** @class */ (function () {
    function IconShare() {
    }
    IconShare.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconShareComponent],
                    exports: [IconShareComponent]
                },] }
    ];
    return IconShare;
}());
export { IconShare };
