// angular-feather/esm5/feather/share-2.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconShare2Component = /** @class */ (function () {
    function IconShare2Component() {
    }
    IconShare2Component.decorators = [
        { type: Component, args: [{
                    selector: 'i-share-2',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-share-2\">\n    <circle cx=\"18\" cy=\"5\" r=\"3\"></circle><circle cx=\"6\" cy=\"12\" r=\"3\"></circle><circle cx=\"18\" cy=\"19\" r=\"3\"></circle><line x1=\"8.59\" y1=\"13.51\" x2=\"15.42\" y2=\"17.49\"></line><line x1=\"15.41\" y1=\"6.51\" x2=\"8.59\" y2=\"10.49\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconShare2Component;
}());
export { IconShare2Component };
var IconShare2 = /** @class */ (function () {
    function IconShare2() {
    }
    IconShare2.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconShare2Component],
                    exports: [IconShare2Component]
                },] }
    ];
    return IconShare2;
}());
export { IconShare2 };
