// angular-feather/esm5/feather/shield-off.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconShieldOffComponent = /** @class */ (function () {
    function IconShieldOffComponent() {
    }
    IconShieldOffComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-shield-off',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-shield-off\">\n    <path d=\"M19.69 14a6.9 6.9 0 0 0 .31-2V5l-8-3-3.16 1.18\"></path><path d=\"M4.73 4.73L4 5v7c0 6 8 10 8 10a20.29 20.29 0 0 0 5.62-4.38\"></path><line x1=\"1\" y1=\"1\" x2=\"23\" y2=\"23\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconShieldOffComponent;
}());
export { IconShieldOffComponent };
var IconShieldOff = /** @class */ (function () {
    function IconShieldOff() {
    }
    IconShieldOff.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconShieldOffComponent],
                    exports: [IconShieldOffComponent]
                },] }
    ];
    return IconShieldOff;
}());
export { IconShieldOff };
