// angular-feather/esm5/feather/shield.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconShieldComponent = /** @class */ (function () {
    function IconShieldComponent() {
    }
    IconShieldComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-shield',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-shield\">\n    <path d=\"M12 22s8-4 8-10V5l-8-3-8 3v7c0 6 8 10 8 10z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconShieldComponent;
}());
export { IconShieldComponent };
var IconShield = /** @class */ (function () {
    function IconShield() {
    }
    IconShield.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconShieldComponent],
                    exports: [IconShieldComponent]
                },] }
    ];
    return IconShield;
}());
export { IconShield };
