// angular-feather/esm5/feather/shopping-cart.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconShoppingCartComponent = /** @class */ (function () {
    function IconShoppingCartComponent() {
    }
    IconShoppingCartComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-shopping-cart',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-shopping-cart\">\n    <circle cx=\"9\" cy=\"21\" r=\"1\"></circle><circle cx=\"20\" cy=\"21\" r=\"1\"></circle><path d=\"M1 1h4l2.68 13.39a2 2 0 0 0 2 1.61h9.72a2 2 0 0 0 2-1.61L23 6H6\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconShoppingCartComponent;
}());
export { IconShoppingCartComponent };
var IconShoppingCart = /** @class */ (function () {
    function IconShoppingCart() {
    }
    IconShoppingCart.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconShoppingCartComponent],
                    exports: [IconShoppingCartComponent]
                },] }
    ];
    return IconShoppingCart;
}());
export { IconShoppingCart };
