// angular-feather/esm5/feather/shopping-bag.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconShoppingBagComponent = /** @class */ (function () {
    function IconShoppingBagComponent() {
    }
    IconShoppingBagComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-shopping-bag',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-shopping-bag\">\n    <path d=\"M6 2L3 6v14a2 2 0 0 0 2 2h14a2 2 0 0 0 2-2V6l-3-4z\"></path><line x1=\"3\" y1=\"6\" x2=\"21\" y2=\"6\"></line><path d=\"M16 10a4 4 0 0 1-8 0\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconShoppingBagComponent;
}());
export { IconShoppingBagComponent };
var IconShoppingBag = /** @class */ (function () {
    function IconShoppingBag() {
    }
    IconShoppingBag.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconShoppingBagComponent],
                    exports: [IconShoppingBagComponent]
                },] }
    ];
    return IconShoppingBag;
}());
export { IconShoppingBag };
