// angular-feather/esm5/feather/shuffle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconShuffleComponent = /** @class */ (function () {
    function IconShuffleComponent() {
    }
    IconShuffleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-shuffle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-shuffle\">\n    <polyline points=\"16 3 21 3 21 8\"></polyline><line x1=\"4\" y1=\"20\" x2=\"21\" y2=\"3\"></line><polyline points=\"21 16 21 21 16 21\"></polyline><line x1=\"15\" y1=\"15\" x2=\"21\" y2=\"21\"></line><line x1=\"4\" y1=\"4\" x2=\"9\" y2=\"9\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconShuffleComponent;
}());
export { IconShuffleComponent };
var IconShuffle = /** @class */ (function () {
    function IconShuffle() {
    }
    IconShuffle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconShuffleComponent],
                    exports: [IconShuffleComponent]
                },] }
    ];
    return IconShuffle;
}());
export { IconShuffle };
