// angular-feather/esm5/feather/sidebar.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconSidebarComponent = /** @class */ (function () {
    function IconSidebarComponent() {
    }
    IconSidebarComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-sidebar',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-sidebar\">\n    <rect x=\"3\" y=\"3\" width=\"18\" height=\"18\" rx=\"2\" ry=\"2\"></rect><line x1=\"9\" y1=\"3\" x2=\"9\" y2=\"21\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconSidebarComponent;
}());
export { IconSidebarComponent };
var IconSidebar = /** @class */ (function () {
    function IconSidebar() {
    }
    IconSidebar.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconSidebarComponent],
                    exports: [IconSidebarComponent]
                },] }
    ];
    return IconSidebar;
}());
export { IconSidebar };
