// angular-feather/esm5/feather/skip-back.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconSkipBackComponent = /** @class */ (function () {
    function IconSkipBackComponent() {
    }
    IconSkipBackComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-skip-back',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-skip-back\">\n    <polygon points=\"19 20 9 12 19 4 19 20\"></polygon><line x1=\"5\" y1=\"19\" x2=\"5\" y2=\"5\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconSkipBackComponent;
}());
export { IconSkipBackComponent };
var IconSkipBack = /** @class */ (function () {
    function IconSkipBack() {
    }
    IconSkipBack.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconSkipBackComponent],
                    exports: [IconSkipBackComponent]
                },] }
    ];
    return IconSkipBack;
}());
export { IconSkipBack };
