// angular-feather/esm5/feather/skip-forward.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconSkipForwardComponent = /** @class */ (function () {
    function IconSkipForwardComponent() {
    }
    IconSkipForwardComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-skip-forward',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-skip-forward\">\n    <polygon points=\"5 4 15 12 5 20 5 4\"></polygon><line x1=\"19\" y1=\"5\" x2=\"19\" y2=\"19\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconSkipForwardComponent;
}());
export { IconSkipForwardComponent };
var IconSkipForward = /** @class */ (function () {
    function IconSkipForward() {
    }
    IconSkipForward.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconSkipForwardComponent],
                    exports: [IconSkipForwardComponent]
                },] }
    ];
    return IconSkipForward;
}());
export { IconSkipForward };
