// angular-feather/esm5/feather/slash.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconSlashComponent = /** @class */ (function () {
    function IconSlashComponent() {
    }
    IconSlashComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-slash',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-slash\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><line x1=\"4.93\" y1=\"4.93\" x2=\"19.07\" y2=\"19.07\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconSlashComponent;
}());
export { IconSlashComponent };
var IconSlash = /** @class */ (function () {
    function IconSlash() {
    }
    IconSlash.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconSlashComponent],
                    exports: [IconSlashComponent]
                },] }
    ];
    return IconSlash;
}());
export { IconSlash };
