// angular-feather/esm5/feather/smartphone.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconSmartphoneComponent = /** @class */ (function () {
    function IconSmartphoneComponent() {
    }
    IconSmartphoneComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-smartphone',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-smartphone\">\n    <rect x=\"5\" y=\"2\" width=\"14\" height=\"20\" rx=\"2\" ry=\"2\"></rect><line x1=\"12\" y1=\"18\" x2=\"12\" y2=\"18\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconSmartphoneComponent;
}());
export { IconSmartphoneComponent };
var IconSmartphone = /** @class */ (function () {
    function IconSmartphone() {
    }
    IconSmartphone.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconSmartphoneComponent],
                    exports: [IconSmartphoneComponent]
                },] }
    ];
    return IconSmartphone;
}());
export { IconSmartphone };
