// angular-feather/esm5/feather/smile.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconSmileComponent = /** @class */ (function () {
    function IconSmileComponent() {
    }
    IconSmileComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-smile',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-smile\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><path d=\"M8 14s1.5 2 4 2 4-2 4-2\"></path><line x1=\"9\" y1=\"9\" x2=\"9.01\" y2=\"9\"></line><line x1=\"15\" y1=\"9\" x2=\"15.01\" y2=\"9\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconSmileComponent;
}());
export { IconSmileComponent };
var IconSmile = /** @class */ (function () {
    function IconSmile() {
    }
    IconSmile.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconSmileComponent],
                    exports: [IconSmileComponent]
                },] }
    ];
    return IconSmile;
}());
export { IconSmile };
