// angular-feather/esm5/feather/speaker.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconSpeakerComponent = /** @class */ (function () {
    function IconSpeakerComponent() {
    }
    IconSpeakerComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-speaker',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-speaker\">\n    <rect x=\"4\" y=\"2\" width=\"16\" height=\"20\" rx=\"2\" ry=\"2\"></rect><circle cx=\"12\" cy=\"14\" r=\"4\"></circle><line x1=\"12\" y1=\"6\" x2=\"12\" y2=\"6\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconSpeakerComponent;
}());
export { IconSpeakerComponent };
var IconSpeaker = /** @class */ (function () {
    function IconSpeaker() {
    }
    IconSpeaker.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconSpeakerComponent],
                    exports: [IconSpeakerComponent]
                },] }
    ];
    return IconSpeaker;
}());
export { IconSpeaker };
