// angular-feather/esm5/feather/square.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconSquareComponent = /** @class */ (function () {
    function IconSquareComponent() {
    }
    IconSquareComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-square',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-square\">\n    <rect x=\"3\" y=\"3\" width=\"18\" height=\"18\" rx=\"2\" ry=\"2\"></rect>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconSquareComponent;
}());
export { IconSquareComponent };
var IconSquare = /** @class */ (function () {
    function IconSquare() {
    }
    IconSquare.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconSquareComponent],
                    exports: [IconSquareComponent]
                },] }
    ];
    return IconSquare;
}());
export { IconSquare };
