// angular-feather/esm5/feather/star.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconStarComponent = /** @class */ (function () {
    function IconStarComponent() {
    }
    IconStarComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-star',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-star\">\n    <polygon points=\"12 2 15.09 8.26 22 9.27 17 14.14 18.18 21.02 12 17.77 5.82 21.02 7 14.14 2 9.27 8.91 8.26 12 2\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconStarComponent;
}());
export { IconStarComponent };
var IconStar = /** @class */ (function () {
    function IconStar() {
    }
    IconStar.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconStarComponent],
                    exports: [IconStarComponent]
                },] }
    ];
    return IconStar;
}());
export { IconStar };
