// angular-feather/esm5/feather/stop-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconStopCircleComponent = /** @class */ (function () {
    function IconStopCircleComponent() {
    }
    IconStopCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-stop-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-stop-circle\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><rect x=\"9\" y=\"9\" width=\"6\" height=\"6\"></rect>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconStopCircleComponent;
}());
export { IconStopCircleComponent };
var IconStopCircle = /** @class */ (function () {
    function IconStopCircle() {
    }
    IconStopCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconStopCircleComponent],
                    exports: [IconStopCircleComponent]
                },] }
    ];
    return IconStopCircle;
}());
export { IconStopCircle };
