// angular-feather/esm5/feather/tablet.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTabletComponent = /** @class */ (function () {
    function IconTabletComponent() {
    }
    IconTabletComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-tablet',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-tablet\">\n    <rect x=\"4\" y=\"2\" width=\"16\" height=\"20\" rx=\"2\" ry=\"2\" transform=\"rotate(180 12 12)\"></rect><line x1=\"12\" y1=\"18\" x2=\"12\" y2=\"18\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTabletComponent;
}());
export { IconTabletComponent };
var IconTablet = /** @class */ (function () {
    function IconTablet() {
    }
    IconTablet.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTabletComponent],
                    exports: [IconTabletComponent]
                },] }
    ];
    return IconTablet;
}());
export { IconTablet };
