// angular-feather/esm5/feather/tag.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTagComponent = /** @class */ (function () {
    function IconTagComponent() {
    }
    IconTagComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-tag',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-tag\">\n    <path d=\"M20.59 13.41l-7.17 7.17a2 2 0 0 1-2.83 0L2 12V2h10l8.59 8.59a2 2 0 0 1 0 2.82z\"></path><line x1=\"7\" y1=\"7\" x2=\"7\" y2=\"7\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTagComponent;
}());
export { IconTagComponent };
var IconTag = /** @class */ (function () {
    function IconTag() {
    }
    IconTag.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTagComponent],
                    exports: [IconTagComponent]
                },] }
    ];
    return IconTag;
}());
export { IconTag };
