// angular-feather/esm5/feather/target.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTargetComponent = /** @class */ (function () {
    function IconTargetComponent() {
    }
    IconTargetComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-target',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-target\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><circle cx=\"12\" cy=\"12\" r=\"6\"></circle><circle cx=\"12\" cy=\"12\" r=\"2\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTargetComponent;
}());
export { IconTargetComponent };
var IconTarget = /** @class */ (function () {
    function IconTarget() {
    }
    IconTarget.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTargetComponent],
                    exports: [IconTargetComponent]
                },] }
    ];
    return IconTarget;
}());
export { IconTarget };
