// angular-feather/esm5/feather/terminal.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTerminalComponent = /** @class */ (function () {
    function IconTerminalComponent() {
    }
    IconTerminalComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-terminal',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-terminal\">\n    <polyline points=\"4 17 10 11 4 5\"></polyline><line x1=\"12\" y1=\"19\" x2=\"20\" y2=\"19\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTerminalComponent;
}());
export { IconTerminalComponent };
var IconTerminal = /** @class */ (function () {
    function IconTerminal() {
    }
    IconTerminal.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTerminalComponent],
                    exports: [IconTerminalComponent]
                },] }
    ];
    return IconTerminal;
}());
export { IconTerminal };
