// angular-feather/esm5/feather/thermometer.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconThermometerComponent = /** @class */ (function () {
    function IconThermometerComponent() {
    }
    IconThermometerComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-thermometer',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-thermometer\">\n    <path d=\"M14 14.76V3.5a2.5 2.5 0 0 0-5 0v11.26a4.5 4.5 0 1 0 5 0z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconThermometerComponent;
}());
export { IconThermometerComponent };
var IconThermometer = /** @class */ (function () {
    function IconThermometer() {
    }
    IconThermometer.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconThermometerComponent],
                    exports: [IconThermometerComponent]
                },] }
    ];
    return IconThermometer;
}());
export { IconThermometer };
