// angular-feather/esm5/feather/thumbs-up.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconThumbsUpComponent = /** @class */ (function () {
    function IconThumbsUpComponent() {
    }
    IconThumbsUpComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-thumbs-up',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-thumbs-up\">\n    <path d=\"M14 9V5a3 3 0 0 0-3-3l-4 9v11h11.28a2 2 0 0 0 2-1.7l1.38-9a2 2 0 0 0-2-2.3zM7 22H4a2 2 0 0 1-2-2v-7a2 2 0 0 1 2-2h3\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconThumbsUpComponent;
}());
export { IconThumbsUpComponent };
var IconThumbsUp = /** @class */ (function () {
    function IconThumbsUp() {
    }
    IconThumbsUp.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconThumbsUpComponent],
                    exports: [IconThumbsUpComponent]
                },] }
    ];
    return IconThumbsUp;
}());
export { IconThumbsUp };
