// angular-feather/esm5/feather/thumbs-down.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconThumbsDownComponent = /** @class */ (function () {
    function IconThumbsDownComponent() {
    }
    IconThumbsDownComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-thumbs-down',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-thumbs-down\">\n    <path d=\"M10 15v4a3 3 0 0 0 3 3l4-9V2H5.72a2 2 0 0 0-2 1.7l-1.38 9a2 2 0 0 0 2 2.3zm7-13h2.67A2.31 2.31 0 0 1 22 4v7a2.31 2.31 0 0 1-2.33 2H17\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconThumbsDownComponent;
}());
export { IconThumbsDownComponent };
var IconThumbsDown = /** @class */ (function () {
    function IconThumbsDown() {
    }
    IconThumbsDown.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconThumbsDownComponent],
                    exports: [IconThumbsDownComponent]
                },] }
    ];
    return IconThumbsDown;
}());
export { IconThumbsDown };
