// angular-feather/esm5/feather/toggle-left.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconToggleLeftComponent = /** @class */ (function () {
    function IconToggleLeftComponent() {
    }
    IconToggleLeftComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-toggle-left',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-toggle-left\">\n    <rect x=\"1\" y=\"5\" width=\"22\" height=\"14\" rx=\"7\" ry=\"7\"></rect><circle cx=\"8\" cy=\"12\" r=\"3\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconToggleLeftComponent;
}());
export { IconToggleLeftComponent };
var IconToggleLeft = /** @class */ (function () {
    function IconToggleLeft() {
    }
    IconToggleLeft.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconToggleLeftComponent],
                    exports: [IconToggleLeftComponent]
                },] }
    ];
    return IconToggleLeft;
}());
export { IconToggleLeft };
