// angular-feather/esm5/feather/toggle-right.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconToggleRightComponent = /** @class */ (function () {
    function IconToggleRightComponent() {
    }
    IconToggleRightComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-toggle-right',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-toggle-right\">\n    <rect x=\"1\" y=\"5\" width=\"22\" height=\"14\" rx=\"7\" ry=\"7\"></rect><circle cx=\"16\" cy=\"12\" r=\"3\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconToggleRightComponent;
}());
export { IconToggleRightComponent };
var IconToggleRight = /** @class */ (function () {
    function IconToggleRight() {
    }
    IconToggleRight.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconToggleRightComponent],
                    exports: [IconToggleRightComponent]
                },] }
    ];
    return IconToggleRight;
}());
export { IconToggleRight };
