// angular-feather/esm5/feather/trash.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTrashComponent = /** @class */ (function () {
    function IconTrashComponent() {
    }
    IconTrashComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-trash',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-trash\">\n    <polyline points=\"3 6 5 6 21 6\"></polyline><path d=\"M19 6v14a2 2 0 0 1-2 2H7a2 2 0 0 1-2-2V6m3 0V4a2 2 0 0 1 2-2h4a2 2 0 0 1 2 2v2\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTrashComponent;
}());
export { IconTrashComponent };
var IconTrash = /** @class */ (function () {
    function IconTrash() {
    }
    IconTrash.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTrashComponent],
                    exports: [IconTrashComponent]
                },] }
    ];
    return IconTrash;
}());
export { IconTrash };
