// angular-feather/esm5/feather/trash-2.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTrash2Component = /** @class */ (function () {
    function IconTrash2Component() {
    }
    IconTrash2Component.decorators = [
        { type: Component, args: [{
                    selector: 'i-trash-2',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-trash-2\">\n    <polyline points=\"3 6 5 6 21 6\"></polyline><path d=\"M19 6v14a2 2 0 0 1-2 2H7a2 2 0 0 1-2-2V6m3 0V4a2 2 0 0 1 2-2h4a2 2 0 0 1 2 2v2\"></path><line x1=\"10\" y1=\"11\" x2=\"10\" y2=\"17\"></line><line x1=\"14\" y1=\"11\" x2=\"14\" y2=\"17\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTrash2Component;
}());
export { IconTrash2Component };
var IconTrash2 = /** @class */ (function () {
    function IconTrash2() {
    }
    IconTrash2.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTrash2Component],
                    exports: [IconTrash2Component]
                },] }
    ];
    return IconTrash2;
}());
export { IconTrash2 };
