// angular-feather/esm5/feather/trello.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTrelloComponent = /** @class */ (function () {
    function IconTrelloComponent() {
    }
    IconTrelloComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-trello',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-trello\">\n    <rect x=\"3\" y=\"3\" width=\"18\" height=\"18\" rx=\"2\" ry=\"2\"></rect><rect x=\"7\" y=\"7\" width=\"3\" height=\"9\"></rect><rect x=\"14\" y=\"7\" width=\"3\" height=\"5\"></rect>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTrelloComponent;
}());
export { IconTrelloComponent };
var IconTrello = /** @class */ (function () {
    function IconTrello() {
    }
    IconTrello.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTrelloComponent],
                    exports: [IconTrelloComponent]
                },] }
    ];
    return IconTrello;
}());
export { IconTrello };
