// angular-feather/esm5/feather/trending-down.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTrendingDownComponent = /** @class */ (function () {
    function IconTrendingDownComponent() {
    }
    IconTrendingDownComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-trending-down',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-trending-down\">\n    <polyline points=\"23 18 13.5 8.5 8.5 13.5 1 6\"></polyline><polyline points=\"17 18 23 18 23 12\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTrendingDownComponent;
}());
export { IconTrendingDownComponent };
var IconTrendingDown = /** @class */ (function () {
    function IconTrendingDown() {
    }
    IconTrendingDown.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTrendingDownComponent],
                    exports: [IconTrendingDownComponent]
                },] }
    ];
    return IconTrendingDown;
}());
export { IconTrendingDown };
