// angular-feather/esm5/feather/trending-up.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTrendingUpComponent = /** @class */ (function () {
    function IconTrendingUpComponent() {
    }
    IconTrendingUpComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-trending-up',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-trending-up\">\n    <polyline points=\"23 6 13.5 15.5 8.5 10.5 1 18\"></polyline><polyline points=\"17 6 23 6 23 12\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTrendingUpComponent;
}());
export { IconTrendingUpComponent };
var IconTrendingUp = /** @class */ (function () {
    function IconTrendingUp() {
    }
    IconTrendingUp.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTrendingUpComponent],
                    exports: [IconTrendingUpComponent]
                },] }
    ];
    return IconTrendingUp;
}());
export { IconTrendingUp };
