// angular-feather/esm5/feather/triangle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTriangleComponent = /** @class */ (function () {
    function IconTriangleComponent() {
    }
    IconTriangleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-triangle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-triangle\">\n    <path d=\"M10.29 3.86L1.82 18a2 2 0 0 0 1.71 3h16.94a2 2 0 0 0 1.71-3L13.71 3.86a2 2 0 0 0-3.42 0z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTriangleComponent;
}());
export { IconTriangleComponent };
var IconTriangle = /** @class */ (function () {
    function IconTriangle() {
    }
    IconTriangle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTriangleComponent],
                    exports: [IconTriangleComponent]
                },] }
    ];
    return IconTriangle;
}());
export { IconTriangle };
