// angular-feather/esm5/feather/tv.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTvComponent = /** @class */ (function () {
    function IconTvComponent() {
    }
    IconTvComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-tv',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-tv\">\n    <rect x=\"2\" y=\"7\" width=\"20\" height=\"15\" rx=\"2\" ry=\"2\"></rect><polyline points=\"17 2 12 7 7 2\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTvComponent;
}());
export { IconTvComponent };
var IconTv = /** @class */ (function () {
    function IconTv() {
    }
    IconTv.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTvComponent],
                    exports: [IconTvComponent]
                },] }
    ];
    return IconTv;
}());
export { IconTv };
