// angular-feather/esm5/feather/twitter.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTwitterComponent = /** @class */ (function () {
    function IconTwitterComponent() {
    }
    IconTwitterComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-twitter',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-twitter\">\n    <path d=\"M23 3a10.9 10.9 0 0 1-3.14 1.53 4.48 4.48 0 0 0-7.86 3v1A10.66 10.66 0 0 1 3 4s-4 9 5 13a11.64 11.64 0 0 1-7 2c9 5 20 0 20-11.5a4.5 4.5 0 0 0-.08-.83A7.72 7.72 0 0 0 23 3z\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTwitterComponent;
}());
export { IconTwitterComponent };
var IconTwitter = /** @class */ (function () {
    function IconTwitter() {
    }
    IconTwitter.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTwitterComponent],
                    exports: [IconTwitterComponent]
                },] }
    ];
    return IconTwitter;
}());
export { IconTwitter };
