// angular-feather/esm5/feather/type.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconTypeComponent = /** @class */ (function () {
    function IconTypeComponent() {
    }
    IconTypeComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-type',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-type\">\n    <polyline points=\"4 7 4 4 20 4 20 7\"></polyline><line x1=\"9\" y1=\"20\" x2=\"15\" y2=\"20\"></line><line x1=\"12\" y1=\"4\" x2=\"12\" y2=\"20\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconTypeComponent;
}());
export { IconTypeComponent };
var IconType = /** @class */ (function () {
    function IconType() {
    }
    IconType.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconTypeComponent],
                    exports: [IconTypeComponent]
                },] }
    ];
    return IconType;
}());
export { IconType };
