// angular-feather/esm5/feather/umbrella.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconUmbrellaComponent = /** @class */ (function () {
    function IconUmbrellaComponent() {
    }
    IconUmbrellaComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-umbrella',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-umbrella\">\n    <path d=\"M23 12a11.05 11.05 0 0 0-22 0zm-5 7a3 3 0 0 1-6 0v-7\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconUmbrellaComponent;
}());
export { IconUmbrellaComponent };
var IconUmbrella = /** @class */ (function () {
    function IconUmbrella() {
    }
    IconUmbrella.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconUmbrellaComponent],
                    exports: [IconUmbrellaComponent]
                },] }
    ];
    return IconUmbrella;
}());
export { IconUmbrella };
