// angular-feather/esm5/feather/underline.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconUnderlineComponent = /** @class */ (function () {
    function IconUnderlineComponent() {
    }
    IconUnderlineComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-underline',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-underline\">\n    <path d=\"M6 3v7a6 6 0 0 0 6 6 6 6 0 0 0 6-6V3\"></path><line x1=\"4\" y1=\"21\" x2=\"20\" y2=\"21\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconUnderlineComponent;
}());
export { IconUnderlineComponent };
var IconUnderline = /** @class */ (function () {
    function IconUnderline() {
    }
    IconUnderline.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconUnderlineComponent],
                    exports: [IconUnderlineComponent]
                },] }
    ];
    return IconUnderline;
}());
export { IconUnderline };
