// angular-feather/esm5/feather/upload-cloud.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconUploadCloudComponent = /** @class */ (function () {
    function IconUploadCloudComponent() {
    }
    IconUploadCloudComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-upload-cloud',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-upload-cloud\">\n    <polyline points=\"16 16 12 12 8 16\"></polyline><line x1=\"12\" y1=\"12\" x2=\"12\" y2=\"21\"></line><path d=\"M20.39 18.39A5 5 0 0 0 18 9h-1.26A8 8 0 1 0 3 16.3\"></path><polyline points=\"16 16 12 12 8 16\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconUploadCloudComponent;
}());
export { IconUploadCloudComponent };
var IconUploadCloud = /** @class */ (function () {
    function IconUploadCloud() {
    }
    IconUploadCloud.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconUploadCloudComponent],
                    exports: [IconUploadCloudComponent]
                },] }
    ];
    return IconUploadCloud;
}());
export { IconUploadCloud };
