// angular-feather/esm5/feather/upload.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconUploadComponent = /** @class */ (function () {
    function IconUploadComponent() {
    }
    IconUploadComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-upload',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-upload\">\n    <path d=\"M21 15v4a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-4\"></path><polyline points=\"17 8 12 3 7 8\"></polyline><line x1=\"12\" y1=\"3\" x2=\"12\" y2=\"15\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconUploadComponent;
}());
export { IconUploadComponent };
var IconUpload = /** @class */ (function () {
    function IconUpload() {
    }
    IconUpload.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconUploadComponent],
                    exports: [IconUploadComponent]
                },] }
    ];
    return IconUpload;
}());
export { IconUpload };
