// angular-feather/esm5/feather/user-check.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconUserCheckComponent = /** @class */ (function () {
    function IconUserCheckComponent() {
    }
    IconUserCheckComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-user-check',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-user-check\">\n    <path d=\"M16 21v-2a4 4 0 0 0-4-4H5a4 4 0 0 0-4 4v2\"></path><circle cx=\"8.5\" cy=\"7\" r=\"4\"></circle><polyline points=\"17 11 19 13 23 9\"></polyline>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconUserCheckComponent;
}());
export { IconUserCheckComponent };
var IconUserCheck = /** @class */ (function () {
    function IconUserCheck() {
    }
    IconUserCheck.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconUserCheckComponent],
                    exports: [IconUserCheckComponent]
                },] }
    ];
    return IconUserCheck;
}());
export { IconUserCheck };
