// angular-feather/esm5/feather/user-minus.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconUserMinusComponent = /** @class */ (function () {
    function IconUserMinusComponent() {
    }
    IconUserMinusComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-user-minus',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-user-minus\">\n    <path d=\"M16 21v-2a4 4 0 0 0-4-4H5a4 4 0 0 0-4 4v2\"></path><circle cx=\"8.5\" cy=\"7\" r=\"4\"></circle><line x1=\"23\" y1=\"11\" x2=\"17\" y2=\"11\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconUserMinusComponent;
}());
export { IconUserMinusComponent };
var IconUserMinus = /** @class */ (function () {
    function IconUserMinus() {
    }
    IconUserMinus.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconUserMinusComponent],
                    exports: [IconUserMinusComponent]
                },] }
    ];
    return IconUserMinus;
}());
export { IconUserMinus };
