// angular-feather/esm5/feather/users.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconUsersComponent = /** @class */ (function () {
    function IconUsersComponent() {
    }
    IconUsersComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-users',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-users\">\n    <path d=\"M17 21v-2a4 4 0 0 0-4-4H5a4 4 0 0 0-4 4v2\"></path><circle cx=\"9\" cy=\"7\" r=\"4\"></circle><path d=\"M23 21v-2a4 4 0 0 0-3-3.87\"></path><path d=\"M16 3.13a4 4 0 0 1 0 7.75\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconUsersComponent;
}());
export { IconUsersComponent };
var IconUsers = /** @class */ (function () {
    function IconUsers() {
    }
    IconUsers.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconUsersComponent],
                    exports: [IconUsersComponent]
                },] }
    ];
    return IconUsers;
}());
export { IconUsers };
