// angular-feather/esm5/feather/user.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconUserComponent = /** @class */ (function () {
    function IconUserComponent() {
    }
    IconUserComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-user',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-user\">\n    <path d=\"M20 21v-2a4 4 0 0 0-4-4H8a4 4 0 0 0-4 4v2\"></path><circle cx=\"12\" cy=\"7\" r=\"4\"></circle>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconUserComponent;
}());
export { IconUserComponent };
var IconUser = /** @class */ (function () {
    function IconUser() {
    }
    IconUser.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconUserComponent],
                    exports: [IconUserComponent]
                },] }
    ];
    return IconUser;
}());
export { IconUser };
