// angular-feather/esm5/feather/video.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconVideoComponent = /** @class */ (function () {
    function IconVideoComponent() {
    }
    IconVideoComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-video',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-video\">\n    <polygon points=\"23 7 16 12 23 17 23 7\"></polygon><rect x=\"1\" y=\"5\" width=\"15\" height=\"14\" rx=\"2\" ry=\"2\"></rect>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconVideoComponent;
}());
export { IconVideoComponent };
var IconVideo = /** @class */ (function () {
    function IconVideo() {
    }
    IconVideo.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconVideoComponent],
                    exports: [IconVideoComponent]
                },] }
    ];
    return IconVideo;
}());
export { IconVideo };
