// angular-feather/esm5/feather/video-off.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconVideoOffComponent = /** @class */ (function () {
    function IconVideoOffComponent() {
    }
    IconVideoOffComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-video-off',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-video-off\">\n    <path d=\"M16 16v1a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V7a2 2 0 0 1 2-2h2m5.66 0H14a2 2 0 0 1 2 2v3.34l1 1L23 7v10\"></path><line x1=\"1\" y1=\"1\" x2=\"23\" y2=\"23\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconVideoOffComponent;
}());
export { IconVideoOffComponent };
var IconVideoOff = /** @class */ (function () {
    function IconVideoOff() {
    }
    IconVideoOff.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconVideoOffComponent],
                    exports: [IconVideoOffComponent]
                },] }
    ];
    return IconVideoOff;
}());
export { IconVideoOff };
