// angular-feather/esm5/feather/voicemail.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconVoicemailComponent = /** @class */ (function () {
    function IconVoicemailComponent() {
    }
    IconVoicemailComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-voicemail',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-voicemail\">\n    <circle cx=\"5.5\" cy=\"11.5\" r=\"4.5\"></circle><circle cx=\"18.5\" cy=\"11.5\" r=\"4.5\"></circle><line x1=\"5.5\" y1=\"16\" x2=\"18.5\" y2=\"16\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconVoicemailComponent;
}());
export { IconVoicemailComponent };
var IconVoicemail = /** @class */ (function () {
    function IconVoicemail() {
    }
    IconVoicemail.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconVoicemailComponent],
                    exports: [IconVoicemailComponent]
                },] }
    ];
    return IconVoicemail;
}());
export { IconVoicemail };
