// angular-feather/esm5/feather/volume-1.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconVolume1Component = /** @class */ (function () {
    function IconVolume1Component() {
    }
    IconVolume1Component.decorators = [
        { type: Component, args: [{
                    selector: 'i-volume-1',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-volume-1\">\n    <polygon points=\"11 5 6 9 2 9 2 15 6 15 11 19 11 5\"></polygon><path d=\"M15.54 8.46a5 5 0 0 1 0 7.07\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconVolume1Component;
}());
export { IconVolume1Component };
var IconVolume1 = /** @class */ (function () {
    function IconVolume1() {
    }
    IconVolume1.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconVolume1Component],
                    exports: [IconVolume1Component]
                },] }
    ];
    return IconVolume1;
}());
export { IconVolume1 };
