// angular-feather/esm5/feather/volume-x.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconVolumeXComponent = /** @class */ (function () {
    function IconVolumeXComponent() {
    }
    IconVolumeXComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-volume-x',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-volume-x\">\n    <polygon points=\"11 5 6 9 2 9 2 15 6 15 11 19 11 5\"></polygon><line x1=\"23\" y1=\"9\" x2=\"17\" y2=\"15\"></line><line x1=\"17\" y1=\"9\" x2=\"23\" y2=\"15\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconVolumeXComponent;
}());
export { IconVolumeXComponent };
var IconVolumeX = /** @class */ (function () {
    function IconVolumeX() {
    }
    IconVolumeX.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconVolumeXComponent],
                    exports: [IconVolumeXComponent]
                },] }
    ];
    return IconVolumeX;
}());
export { IconVolumeX };
