// angular-feather/esm5/feather/volume.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconVolumeComponent = /** @class */ (function () {
    function IconVolumeComponent() {
    }
    IconVolumeComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-volume',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-volume\">\n    <polygon points=\"11 5 6 9 2 9 2 15 6 15 11 19 11 5\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconVolumeComponent;
}());
export { IconVolumeComponent };
var IconVolume = /** @class */ (function () {
    function IconVolume() {
    }
    IconVolume.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconVolumeComponent],
                    exports: [IconVolumeComponent]
                },] }
    ];
    return IconVolume;
}());
export { IconVolume };
