// angular-feather/esm5/feather/watch.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconWatchComponent = /** @class */ (function () {
    function IconWatchComponent() {
    }
    IconWatchComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-watch',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-watch\">\n    <circle cx=\"12\" cy=\"12\" r=\"7\"></circle><polyline points=\"12 9 12 12 13.5 13.5\"></polyline><path d=\"M16.51 17.35l-.35 3.83a2 2 0 0 1-2 1.82H9.83a2 2 0 0 1-2-1.82l-.35-3.83m.01-10.7l.35-3.83A2 2 0 0 1 9.83 1h4.35a2 2 0 0 1 2 1.82l.35 3.83\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconWatchComponent;
}());
export { IconWatchComponent };
var IconWatch = /** @class */ (function () {
    function IconWatch() {
    }
    IconWatch.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconWatchComponent],
                    exports: [IconWatchComponent]
                },] }
    ];
    return IconWatch;
}());
export { IconWatch };
