// angular-feather/esm5/feather/wifi.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconWifiComponent = /** @class */ (function () {
    function IconWifiComponent() {
    }
    IconWifiComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-wifi',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-wifi\">\n    <path d=\"M5 12.55a11 11 0 0 1 14.08 0\"></path><path d=\"M1.42 9a16 16 0 0 1 21.16 0\"></path><path d=\"M8.53 16.11a6 6 0 0 1 6.95 0\"></path><line x1=\"12\" y1=\"20\" x2=\"12\" y2=\"20\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconWifiComponent;
}());
export { IconWifiComponent };
var IconWifi = /** @class */ (function () {
    function IconWifi() {
    }
    IconWifi.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconWifiComponent],
                    exports: [IconWifiComponent]
                },] }
    ];
    return IconWifi;
}());
export { IconWifi };
