// angular-feather/esm5/feather/wifi-off.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconWifiOffComponent = /** @class */ (function () {
    function IconWifiOffComponent() {
    }
    IconWifiOffComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-wifi-off',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-wifi-off\">\n    <line x1=\"1\" y1=\"1\" x2=\"23\" y2=\"23\"></line><path d=\"M16.72 11.06A10.94 10.94 0 0 1 19 12.55\"></path><path d=\"M5 12.55a10.94 10.94 0 0 1 5.17-2.39\"></path><path d=\"M10.71 5.05A16 16 0 0 1 22.58 9\"></path><path d=\"M1.42 9a15.91 15.91 0 0 1 4.7-2.88\"></path><path d=\"M8.53 16.11a6 6 0 0 1 6.95 0\"></path><line x1=\"12\" y1=\"20\" x2=\"12\" y2=\"20\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconWifiOffComponent;
}());
export { IconWifiOffComponent };
var IconWifiOff = /** @class */ (function () {
    function IconWifiOff() {
    }
    IconWifiOff.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconWifiOffComponent],
                    exports: [IconWifiOffComponent]
                },] }
    ];
    return IconWifiOff;
}());
export { IconWifiOff };
