// angular-feather/esm5/feather/wind.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconWindComponent = /** @class */ (function () {
    function IconWindComponent() {
    }
    IconWindComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-wind',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-wind\">\n    <path d=\"M9.59 4.59A2 2 0 1 1 11 8H2m10.59 11.41A2 2 0 1 0 14 16H2m15.73-8.27A2.5 2.5 0 1 1 19.5 12H2\"></path>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconWindComponent;
}());
export { IconWindComponent };
var IconWind = /** @class */ (function () {
    function IconWind() {
    }
    IconWind.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconWindComponent],
                    exports: [IconWindComponent]
                },] }
    ];
    return IconWind;
}());
export { IconWind };
