// angular-feather/esm5/feather/x-circle.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconXCircleComponent = /** @class */ (function () {
    function IconXCircleComponent() {
    }
    IconXCircleComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-x-circle',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-x-circle\">\n    <circle cx=\"12\" cy=\"12\" r=\"10\"></circle><line x1=\"15\" y1=\"9\" x2=\"9\" y2=\"15\"></line><line x1=\"9\" y1=\"9\" x2=\"15\" y2=\"15\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconXCircleComponent;
}());
export { IconXCircleComponent };
var IconXCircle = /** @class */ (function () {
    function IconXCircle() {
    }
    IconXCircle.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconXCircleComponent],
                    exports: [IconXCircleComponent]
                },] }
    ];
    return IconXCircle;
}());
export { IconXCircle };
