// angular-feather/esm5/feather/x-octagon.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconXOctagonComponent = /** @class */ (function () {
    function IconXOctagonComponent() {
    }
    IconXOctagonComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-x-octagon',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-x-octagon\">\n    <polygon points=\"7.86 2 16.14 2 22 7.86 22 16.14 16.14 22 7.86 22 2 16.14 2 7.86 7.86 2\"></polygon><line x1=\"15\" y1=\"9\" x2=\"9\" y2=\"15\"></line><line x1=\"9\" y1=\"9\" x2=\"15\" y2=\"15\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconXOctagonComponent;
}());
export { IconXOctagonComponent };
var IconXOctagon = /** @class */ (function () {
    function IconXOctagon() {
    }
    IconXOctagon.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconXOctagonComponent],
                    exports: [IconXOctagonComponent]
                },] }
    ];
    return IconXOctagon;
}());
export { IconXOctagon };
