// angular-feather/esm5/feather/x-square.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconXSquareComponent = /** @class */ (function () {
    function IconXSquareComponent() {
    }
    IconXSquareComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-x-square',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-x-square\">\n    <rect x=\"3\" y=\"3\" width=\"18\" height=\"18\" rx=\"2\" ry=\"2\"></rect><line x1=\"9\" y1=\"9\" x2=\"15\" y2=\"15\"></line><line x1=\"15\" y1=\"9\" x2=\"9\" y2=\"15\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconXSquareComponent;
}());
export { IconXSquareComponent };
var IconXSquare = /** @class */ (function () {
    function IconXSquare() {
    }
    IconXSquare.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconXSquareComponent],
                    exports: [IconXSquareComponent]
                },] }
    ];
    return IconXSquare;
}());
export { IconXSquare };
