// angular-feather/esm5/feather/x.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconXComponent = /** @class */ (function () {
    function IconXComponent() {
    }
    IconXComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-x',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-x\">\n    <line x1=\"18\" y1=\"6\" x2=\"6\" y2=\"18\"></line><line x1=\"6\" y1=\"6\" x2=\"18\" y2=\"18\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconXComponent;
}());
export { IconXComponent };
var IconX = /** @class */ (function () {
    function IconX() {
    }
    IconX.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconXComponent],
                    exports: [IconXComponent]
                },] }
    ];
    return IconX;
}());
export { IconX };
