// angular-feather/esm5/feather/youtube.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconYoutubeComponent = /** @class */ (function () {
    function IconYoutubeComponent() {
    }
    IconYoutubeComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-youtube',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-youtube\">\n    <path d=\"M22.54 6.42a2.78 2.78 0 0 0-1.94-2C18.88 4 12 4 12 4s-6.88 0-8.6.46a2.78 2.78 0 0 0-1.94 2A29 29 0 0 0 1 11.75a29 29 0 0 0 .46 5.33A2.78 2.78 0 0 0 3.4 19c1.72.46 8.6.46 8.6.46s6.88 0 8.6-.46a2.78 2.78 0 0 0 1.94-2 29 29 0 0 0 .46-5.25 29 29 0 0 0-.46-5.33z\"></path><polygon points=\"9.75 15.02 15.5 11.75 9.75 8.48 9.75 15.02\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconYoutubeComponent;
}());
export { IconYoutubeComponent };
var IconYoutube = /** @class */ (function () {
    function IconYoutube() {
    }
    IconYoutube.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconYoutubeComponent],
                    exports: [IconYoutubeComponent]
                },] }
    ];
    return IconYoutube;
}());
export { IconYoutube };
