// angular-feather/esm5/feather/zap-off.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconZapOffComponent = /** @class */ (function () {
    function IconZapOffComponent() {
    }
    IconZapOffComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-zap-off',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-zap-off\">\n    <polyline points=\"12.41 6.75 13 2 10.57 4.92\"></polyline><polyline points=\"18.57 12.91 21 10 15.66 10\"></polyline><polyline points=\"8 8 3 14 12 14 11 22 16 16\"></polyline><line x1=\"1\" y1=\"1\" x2=\"23\" y2=\"23\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconZapOffComponent;
}());
export { IconZapOffComponent };
var IconZapOff = /** @class */ (function () {
    function IconZapOff() {
    }
    IconZapOff.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconZapOffComponent],
                    exports: [IconZapOffComponent]
                },] }
    ];
    return IconZapOff;
}());
export { IconZapOff };
