// angular-feather/esm5/feather/zap.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconZapComponent = /** @class */ (function () {
    function IconZapComponent() {
    }
    IconZapComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-zap',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-zap\">\n    <polygon points=\"13 2 3 14 12 14 11 22 21 10 12 10 13 2\"></polygon>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconZapComponent;
}());
export { IconZapComponent };
var IconZap = /** @class */ (function () {
    function IconZap() {
    }
    IconZap.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconZapComponent],
                    exports: [IconZapComponent]
                },] }
    ];
    return IconZap;
}());
export { IconZap };
