// angular-feather/esm5/feather/zoom-out.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconZoomOutComponent = /** @class */ (function () {
    function IconZoomOutComponent() {
    }
    IconZoomOutComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-zoom-out',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-zoom-out\">\n    <circle cx=\"11\" cy=\"11\" r=\"8\"></circle><line x1=\"21\" y1=\"21\" x2=\"16.65\" y2=\"16.65\"></line><line x1=\"8\" y1=\"11\" x2=\"14\" y2=\"11\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconZoomOutComponent;
}());
export { IconZoomOutComponent };
var IconZoomOut = /** @class */ (function () {
    function IconZoomOut() {
    }
    IconZoomOut.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconZoomOutComponent],
                    exports: [IconZoomOutComponent]
                },] }
    ];
    return IconZoomOut;
}());
export { IconZoomOut };
