// angular-feather/esm5/feather/zoom-in.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
import { NgModule } from '@angular/core';
var IconZoomInComponent = /** @class */ (function () {
    function IconZoomInComponent() {
    }
    IconZoomInComponent.decorators = [
        { type: Component, args: [{
                    selector: 'i-zoom-in',
                    template: "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 24 24\" class=\"feather feather-zoom-in\">\n    <circle cx=\"11\" cy=\"11\" r=\"8\"></circle><line x1=\"21\" y1=\"21\" x2=\"16.65\" y2=\"16.65\"></line><line x1=\"11\" y1=\"8\" x2=\"11\" y2=\"14\"></line><line x1=\"8\" y1=\"11\" x2=\"14\" y2=\"11\"></line>\n  </svg>",
                    styles: ["\n    :host {\n      display: inline-block;\n      width: 24px;\n      height: 24px;\n      fill: none;\n      stroke: currentColor;\n      stroke-width: 2px;\n      stroke-linecap: round;\n      stroke-linejoin: round;\n    }\n  "]
                }] }
    ];
    return IconZoomInComponent;
}());
export { IconZoomInComponent };
var IconZoomIn = /** @class */ (function () {
    function IconZoomIn() {
    }
    IconZoomIn.decorators = [
        { type: NgModule, args: [{
                    declarations: [IconZoomInComponent],
                    exports: [IconZoomInComponent]
                },] }
    ];
    return IconZoomIn;
}());
export { IconZoomIn };
